<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
  <title>HomePage of Dr. Tao Zeng (SIBS)</title>


  <style type="text/css">
<!--
.STYLE1 {
font-size: 18px;
font-weight: bold;
}
.STYLE3 {
font-size: 12px;
font-family: "Times New Roman", Times, serif;
}
.STYLE4 {
font-size: 14px;
font-family: "Times New Roman", Times, serif;
}
.STYLE8 {
font-size: 24px;
font-weight: bold;
font-family: "Times New Roman", Times, serif;
}
.STYLE9 {font-size: 24px}
.STYLE10 {font-family: "Times New Roman", Times, serif}
.STYLE11 {font-weight: bold; font-size: 24px;}
.STYLE12 {font-size: 12px}
-->
  </style>
  <style type="text/css">
<!--
.STYLE1 {
font-size: 18px;
font-weight: bold;
}
.STYLE3 {
font-size: 12px;
font-family: "Times New Roman", Times, serif;
}
.STYLE4 {
font-size: 14px;
font-family: "Times New Roman", Times, serif;
}
.STYLE8 {
font-size: 24px;
font-weight: bold;
font-family: "Times New Roman", Times, serif;
}
.STYLE9 {font-size: 24px}
.STYLE10 {font-family: "Times New Roman", Times, serif}
.STYLE11 {font-weight: bold; font-size: 24px;}
.STYLE12 {font-size: 12px}
-->
  </style>
</head>


<body style="color: rgb(25, 25, 25); background-color: rgb(255, 255, 255);" leftmargin="0" topmargin="0" alink="#0000ff" link="#0000ff" marginheight="0" marginwidth="0" vlink="#0000ff">

<script type="text/javascript" src="http://tajs.qq.com/stats?sId=65927253" charset="UTF-8"></script>&lt;&lt;&lt;&lt;&lt;&lt;&lt;
HEAD
<script type="text/javascript" src="http://tajs.qq.com/stats?sId=65927253" charset="UTF-8"></script>
<div align="center">
<h1><font face="Verdana" size="7">Dr. Tao Zeng</font></h1>

<!-- top menu table -->
<!-- top menu table --><br>

<table cellpadding="5" cellspacing="5">

  <tbody>

    <tr>

      <td valign="top" width="25%"><!-- left box table -->
      <table bgcolor="#ffffff" cellpadding="0" cellspacing="1">

        <tbody>

          <tr>

            <td>
            <table bgcolor="#bbbbbb" width="100%">

              <tbody>

                <tr>

                  <td> <font face="Verdana" size="1">
                  <b>News and Photos</b></font> </td>

                </tr>

              </tbody>
            </table>

            </td>

          </tr>

          <tr>

            <td>
            <table bgcolor="#ffffff" width="100%">

              <tbody>

                <tr>

                  <td> <font face="Verdana" size="1">Dr.
Tao Zeng in FuDan University ...2016.5</font><br>

                  <div style="text-align: justify;"><font face="Verdana" size="1">taken an talk about "network
biomarker" in ShangHai Bioinformatics seminar</font></div>

                  <font face="Verdana" size="1"><br>

Dr.
Tao Zeng in BeiJing ................2016.4</font><br>

                  <div style="text-align: justify;"><font face="Verdana" size="1">served
as the academic secretary of The 557th Session of XIANGSHAN-SCIENCE
Conferences: Big Data, Precision Medicine and Bioinformatics</font><br>

                  </div>

                  <font face="Verdana" size="1"><br>

Dr.
Tao Zeng in ShangHai .............2016.3<br>

                  <br>

                  <img style="width: 243px; height: 333px;" src="images/zt.jpeg"><br>

                  <br>

                  </font><font face="Verdana" size="1"><br>

Dr.
Tao Zeng in Hong Kong.............2018.6<br>

                  <br>

                  <img style="width: 243px; height: 333px;" src="images/zt20180615.jpg"> <br>

                  <br>

                  </font><font face="Verdana" size="1"><br>

Dr.
Tao Zeng in ISB2018 (GuiYang) .............2018.8.20<br>

                  <br>

                  <img style="width: 243px; height: 183px;" src="images/ISB2018-zt.jpg"> <br>

                  </font><font face="Verdana" size="1"><br>

Dr.
Tao Zeng in The 9th National Conference on Bioinformatics and Systems
Biology of China.............2020.10<br>

                  <br>

                  <img style="width: 243px; height: 183px;" src="images/2020-10.jpg"> <br>

                  <br>

                  <br>

                  <br>

                  </font> </td>

                </tr>

              </tbody>
            </table>

            </td>

          </tr>

        </tbody>
      </table>

<!-- left box table --> </td>

      <td valign="top">
      <p class="STYLE10"><span class="STYLE8">Associate
Professor</span><br>

      <span class="STYLE3"> &nbsp;&nbsp;CAS Key
Laboratory of Computational Biology, Bio-Med Big Data Center, <br>

&nbsp;&nbsp; &nbsp;&nbsp; Shanghai
Institute of Nutrition and Health, <br>

&nbsp;&nbsp; &nbsp;&nbsp; &nbsp;&nbsp;
University of Chinese Academy of
Sciences, Chinese Academy of Sciences </span></p>

      <p class="STYLE3"> &nbsp;&nbsp;Present
address (office): Cell building 333, 320 YueYang Road, Shanghai 200031,
China<br>

&nbsp;&nbsp;Phone (office): +86-21-5492-0102<br>

&nbsp;&nbsp;E-mail: zengtao(at)bsbii.cn; zengtao(at)sibs.ac.cn<br>

&nbsp;&nbsp;Sites online: <a href="https://www.researchgate.net/profile/Tao_Zeng4">Researchgate</a>,
      <a href="http://www.researcherid.com/rid/A-1711-2012">Researcher
ID</a>, <a href="http://www.scholarmate.com/profile/taoceng36">ScholarMate</a>
, <a href="https://loop.frontiersin.org/people/552744/overview">Loop</a>
, <a href="https://www.linkedin.com/in/tazeng">LinkedIn</a>,
      <a href="https://orcid.org/0000-0002-0295-3994">ORCID</a>
, <a href="https://scholar.google.com/citations?hl=en&amp;user=9Kq0LLQAAAAJ">Google
scholar</a>
, <a href="https://github.com/ztpub">GitHub</a></p>

      <p></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE8">Education:</span><br>

      <span class="STYLE4">&bull;&nbsp;2006~2010 <a href="http://www.whu.edu.cn/">Wuhan University</a>
, <a href="http://cs.whu.edu.cn/">School of computer</a>,
Received the doctor's degree.<br>

&bull;&nbsp;2003~2006 <a href="http://www.whu.edu.cn/">Wuhan
University</a>, <a href="http://maths.whu.edu.cn/">School
of mathematics
and statistics </a>, Received the master's degree.<br>

&bull;&nbsp;1999~2003 <a href="http://www.whu.edu.cn/">Wuhan
University</a>, <a href="http://maths.whu.edu.cn/">School
of mathematics
and statistics </a>, Received the bachelor's degree.</span></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE8">Research
Interests:</span><br>

      <span class="STYLE4">&nbsp;&nbsp;Bioinformatics,
Computational systems biology,
Machine learning, Graph theory, Precision medicine</span></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE8">Research
and professional experence:</span><br>

      <span class="STYLE4">&bull;&nbsp;2019/7-now,
      <strong>associate Professor</strong>, <a href="http://www.bsbii.cn/">Shanghai Research Center for
Brain Science and Brain-Inspired Intelligence</a> <br>

      <span class="STYLE4">&bull;&nbsp;2014/12-2019/07,
      <strong>associate Professor</strong>, <a href="http://www.sysbio.ac.cn/">Key Laboratory
of Systems Biology </a>, <a href="http://www.sibcb.ac.cn/">Institute
of Biochemistry and Cell Biology </a>, <a href="http://www.sibs.cas.cn/">Shanghai
Institutes for Biological Sciences </a>, <a href="http://english.cas.cn/">Chinese Academy of
Sciences </a><br>

&bull;&nbsp;2013/01-2014/11, <strong>assistant Professor</strong>,
      <a href="http://www.sysbio.ac.cn/">Key Laboratory
of Systems Biology </a>, <a href="http://www.sibs.cas.cn/">Shanghai
Institutes for Biological Sciences </a>, <a href="http://english.cas.cn/">Chinese Academy of
Sciences </a><br>

&bull;&nbsp;2011/01-2012/12, <strong>post-doctor</strong>,
      <a href="http://www.sysbio.ac.cn/">Key Laboratory
of Systems Biology </a>, <a href="http://www.sibs.cas.cn/">Shanghai
Institutes for Biological Sciences </a>, <a href="http://english.cas.cn/">Chinese Academy of
Sciences </a><br>

      </span><span class="STYLE4">&bull;&nbsp;2008/07-2009/12,
      <strong>research-associate</strong>, <a href="http://www3.ntu.edu.sg/birc/">BioInformatics
Research Centre </a>, <a href="http://scse.ntu.edu.sg/Pages/Home.aspx">
School of Computer Engineering </a>, <a href="http://www.ntu.edu.sg/Pages/home.aspx">
Nanyang Technological
University </a>, Singapore</span></span></p>

      <p class="STYLE10"></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE8">Academic
activities:</span><br>

      <span class="STYLE4">&bull;&nbsp;The
academic secretary of <a href="http://www.xssc.ac.cn/xs/showconf.asp?pid=455&amp;tid=4">The
557th Session of XIANGSHAN-SCIENCE Conferences: Big Data, Precision
Medicine and Bioinformatics</a> <br>

&bull;&nbsp;The
organizing chair of <a href="http://sysbio.sibcb.ac.cn/cb/BioSynergy2015.html">Bio-synergy
2015</a> <br>

&bull;&nbsp;The
invited report at School of Mathematical Sciences, Peking University
(2020)<br>

&bull;&nbsp;The
invited report at The 9th National Conference on Bioinformatics and
Systems Biology of China, Shanghai (2020) <br>

&bull;&nbsp;The
invited report at 1st CAS Life Science Forum, Urumqi (2019) <br>

&bull;&nbsp;The
invited report at School of Mathematics, South China University of
Technology (2019)<br>

&bull;&nbsp;The
invited report at The Fourth CCF Bioinformatics Conference, GuangZhou
(CBC 2019)<br>

&bull;&nbsp;The
invited report at School of Mathematics, South China University of
Technology (2018)<br>

&bull;&nbsp;The
invited report at National Tianyuan Northwest Center for Mathematics,
Xi`an Jiaotong University (2018)<br>

&bull;&nbsp;The
Editor of <a href="https://www.frontiersin.org/research-topics/8239/machine-learning-advanced-dynamic-omics-data-analysis-for-precision-medicine">
Research Topics (2018) - Machine learning advanced dynamic omics data
analysis
for precision medicine </a> <br>

&bull;&nbsp;The
Editor of <a href="https://www.frontiersin.org/research-topics/12828/cross-domain-analysis-for-all-of-us-precision-medicine">
Research Topics (2020) - Cross-Domain Analysis for &ldquo;All of
Us&rdquo; Precision Medicine </a> <br>

&bull;&nbsp;The
Editor of <a href="https://www.frontiersin.org/research-topics/15661/rna-modification-and-non-coding-rnas-in-human-disease">
Research Topics (2020) - RNA Modification and Non-Coding RNAs in Human
Disease </a> <br>

&bull;&nbsp;The report on Highlights sessions of <a href="http://www.aporc.org/ISB/2015/">ISB 2015</a> and
      <a href="http://www.aporc.org/ISB/2018/">ISB 2018</a>
      <br>

&bull;&nbsp;The
invited report at Shanghai Bioinformatics Seminar (2016)<br>

&bull;&nbsp;The
invited report at School of Mathematics, South China University of
Technology (2016)<br>

&bull;&nbsp;The invited talk on <a href="https://dl.ccf.org.cn/meeting/loadMeetingDetailById?id=5140164653844480%3F">
YOCSEF (WuHan 2020) </a> <br>

&bull;&nbsp;The reviewers for peer-review journals, such as:
Bioinformatics, GPB,
TCBB, JTB, AIIM, SREP, BMC Bioinformatics, BMC Genomics, BMC Cancer,
BMC Systems Biology,
CBM, MBS, Methods, IET SYB, IJBRA, IJMI, IJDMB, Heliyon, IEEE Access,
Plos CB, Computational and Structural Biotechnology Journal, Journal of
Cellular and Molecular Medicine, Computational Biology and Chemistry,
OMICS: A Journal of Integrative Biology, etc <br>

&bull;&nbsp;The program committee member of <a href="http://www.aporc.org/ISB/2013/">ISB 2013</a>
, <a href="http://www.aporc.org/ISB/2014/">ISB 2014</a>
, <a href="http://www.aporc.org/ISB/2015/">ISB 2015</a>
, <a href="http://www.aporc.org/ISB/2016/">ISB 2016</a>
, <a href="http://www.aporc.org/ISB/2017/">ISB 2017</a>
, <a href="https://ritchielab.psu.edu/index.php/tbc2017/">TBC
2017</a>
, <a href="http://datamining-web.it.uts.edu.au/bioinformatics/committees.html">InCob
2017</a> ,<a href="http://www.aporc.org/ISB/2018/Committees.php">ISB
2018</a> , <a href="http://ic-ic.tongji.edu.cn/2018/index.htm">ICIC
2018</a> , <a href="http://orienta.ugr.es/bibm2018/">BIBM
2018</a> , <a href="http://ic-ic.tongji.edu.cn/2019/index.htm">ICIC
2019</a> , <a href="http://ieeebibm.org/BIBM2019/">BIBM
2019</a> , <a href="http://ieeebibm.org/BIBM2020/index.html">BIBM
2020</a>, <a href="http://ieeebibm.org/BIBM2021/index.html">BIBM
2021</a> </span></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE1"><span class="STYLE9">Projcts and awards:</span><br>

      </span><span class="STYLE4">&bull;&nbsp;NSFC
(31200987, 11871456). (PI) <br>

&bull;&nbsp;The Knowledge Innovation Program of SIBS CAS
(2013KIP218). (PI) <br>

&bull;&nbsp;The Natural Science Foundation of Shanghai
(17ZR1446100). (PI) <br>

&bull;&nbsp;The Opening Foundation of Shanghai Municipal
Science
and Technology Major Project (2017SHZDZX01). (PI) <br>

&bull;&nbsp;The Strategic Priority Research Program of CAS
(XDB13040700). <br>

&bull;&nbsp;The National Key R&amp;D Program - Special
Project on Precision Medicine (2016YFC0903400)). <br>

&bull;&nbsp;The National key research and development program
of China (2017YFA0505500). <br>

&bull;&nbsp;Project from Shanghai cooperative center for
transformation &amp; innovation. (Co-PI) <br>

&bull;&nbsp;Travel grant from BIBM2010 <br>

&bull;&nbsp;Travel grants from IDDF2018, IDDF2019 <br>

&bull;&nbsp;Best paper awards from ISB2013 <br>

&bull;&nbsp;Best paper awards from ISB2014 <br>

&bull;&nbsp;Best paper awards from BIBM2016 <br>

&bull;&nbsp;Best paper awards from InCob2017 <br>

&bull;&nbsp;Best poster awards from <a href="http://www.sysbio2019.org/index.php">1st national
systems biology conference (china,shanghai,2019)</a> <br>

&bull;&nbsp;Poster of Distinction awards from IDDF2021 <br>

      </span></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE1"><span class="STYLE9"><span class="STYLE8">Teaching
area and courses</span>:</span><br>

      </span><span class="STYLE4">Bioinformatics,
2013~2021 in SIBS</span>.</p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE11">Publication:</span></p>

      <p class="STYLE10"><em>Journal paper:</em></p>

      <p class="STYLE10">-------------------------------------------------------2021-----------------------------------------------------<br>

      <span class="STYLE3">[1]&nbsp; </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Yu XT, Chen Z. Applying artificial intelligence in the microbiome for
gastrointestinal diseases: A review. J Gastroenterol Hepatol. 2021
Apr;36(4):832-840. doi: 10.1111/jgh.15503.
(Corresponding author) </span></p>

      <p class="STYLE10"><span class="STYLE3">[2]&nbsp;
      </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Huang T, Lu C. Editorial: Cross-Domain Analysis for "All of Us"
Precision Medicine. Front Genet. 2021 Jul 1;12:713771. doi:
10.3389/fgene.2021.713771. PMID: 34276803; PMCID:
PMC8280781.(Corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[3]&nbsp;
Guo WF, Yu XT, Shi QQ, Liang J, Zhang SW, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
. Performance assessment of sample-specific network control methods for
bulk and single-cell biological data analysis. PLoS Comput Biol. 2021
May 6;17(5):e1008962. doi: 10.1371/journal.pcbi.1008962. Online ahead
of print.(Co-corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[4]&nbsp;
Wang L, Xu YN, Chu CC, Jing Z, Chen Y, Zhang J, Pu M, Mi T, Du Y, Liang
Z, Doraiswamy C, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Wu J, Chen L. Facial Skin Microbiota-Mediated Host Response to
Pollution Stress Revealed by Microbiome Networks of Individual.
mSystems. 2021 Aug 31;6(4):e0031921. doi: 10.1128/mSystems.00319-21.
Epub 2021 Jul 27. PMID: 34313461; PMCID: PMC8407115.(Co-corresponding
author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[5]&nbsp;
Guo WF, Zhang SW, Feng YH, Liang J, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L. Network controllability-based algorithm to target
personalized driver genes for discovering combinatorial drugs of
individual patients. Nucleic Acids Res. 2021 Apr 19;49(7):e37. doi:
10.1093/nar/gkaa1272.
(Co-corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[6]&nbsp;
Zhang C, Zhang H, Ge J, Mi T, Cui X, Tu F, Gu X, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L. Landscape dynamic network biomarker analysis reveals the
tipping point of transcriptome reprogramming to prevent skin
photodamage. J Mol Cell Biol. 2021 Oct 5:mjab060. doi:
10.1093/jmcb/mjab060. Epub ahead of print. PMID: 34609489.
(Co-corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[7]&nbsp;
Chen L, Zhou X, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Pan X, Zhang YH, Huang T, Fang Z, Cai YD. Recognizing Pattern and
Rule of Mutation Signatures Corresponding to Cancer Types. Front Cell
Dev Biol. 2021 Aug 26;9:712931. doi: 10.3389/fcell.2021.712931. PMID:
34513841; PMCID: PMC8427289. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[8]&nbsp;
Zhang YH, Guo W, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang S, Chen L, Gamarra M, Mansour RF, Escorcia-Gutierrez J, Huang
T, Cai YD. Identification of Microbiota Biomarkers With Orthologous
Gene Annotation for Type 2 Diabetes. Front Microbiol. 2021 Jul
9;12:711244. doi: 10.3389/fmicb.2021.711244. PMID: 34305880; PMCID:
PMC8299781. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[9]&nbsp;
Zhang YH, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L, Huang T, Cai YD. Determining protein-protein functional
associations by functional rules based on gene ontology and KEGG
pathway. Biochim Biophys Acta Proteins Proteom. 2021
Jun;1869(6):140621. doi: 10.1016/j.bbapap.2021.140621. Epub 2021 Feb 6.
      </span> </p>

      <p class="STYLE10"><span class="STYLE3">[10]&nbsp;
Yuan F, Li Z, Chen L, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Ding S, Huang T, Cai YD. Identifying the Signatures and
Rules of Circulating Extracellular MicroRNA for Distinguishing Cancer
Subtypes. Front Genet. 2021 Mar 9;12:651610. doi:
10.3389/fgene.2021.651610. eCollection 2021. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[11]&nbsp;
Zhang YH, Li H, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L, Li Z, Huang T, Cai YD. Identifying Transcriptomic Signatures
and Rules for SARS-CoV-2 Infection. Front Cell Dev Biol. 2021 Jan
11;8:627302. doi: 10.3389/fcell.2020.627302. eCollection 2020. </span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[12]&nbsp;
Zhang YH, Li Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L, Li H, Gamarra M, Mansour RF, Escorcia-Gutierrez J, Huang T,
Cai YD. Investigating gene methylation signatures for fetal intolerance
prediction. PLoS One. 2021 Apr 22;16(4):e0250032. doi:
10.1371/journal.pone.0250032. eCollection 2021. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[13]&nbsp;
Pan XY, Li H, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Li Z, Chen L, Huang T, Cai YD. Identification of Protein Subcellular
Localization With Network and Functional Embeddings. Front Genet. 2021
Jan 20;11:626500. doi: 10.3389/fgene.2020.626500. eCollection 2020. </span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[14]&nbsp;
Chen L, Li Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Li H, Huang T, Cai YD. Predicting gene phenotype by
multi-label multi-class model based on essential functional features.
Mol Genet Genomics. 2021 Apr 29. doi: 10.1007/s00438-021-01789-8.
Online ahead of print. </span></p>

      <p class="STYLE10"><span class="STYLE3">[15]&nbsp;
Zhang YH, Li Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L, Li H, Huang T, Cai YD. Detecting the Multiomics Signatures of
Factor-Specific Inflammatory Effects on Airway Smooth Muscles. Front
Genet. 2021 Jan 13;11:599970. doi: 10.3389/fgene.2020.599970.
eCollection 2020. </span></p>

      <p class="STYLE10"><span class="STYLE3">[16]&nbsp;
Chen L, Li Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Feng K, Huang T, Cai YD. Identifying
COVID-19-Specific Transcriptomic Biomarkers with Machine Learning
Methods. Biomed Res Int. 2021 Jul 6;2021:9939134. doi:
10.1155/2021/9939134.
PMID: 34307679; PMCID: PMC8272456. </span></p>

      <p class="STYLE10"><span class="STYLE3">[17]&nbsp;
Zhang YH, Li Z, Lu L, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Chen L, Li H, Huang T, Cai YD. Analysis of the Sequence Characteristics
of Antifreeze Protein. Life (Basel). 2021 Jun 3;11(6):520. doi:
10.3390/life11060520. PMID: 34204983; PMCID: PMC8226703.
      </span></p>

      <p class="STYLE10">-------------------------------------------------------2020-----------------------------------------------------<br>

      <span class="STYLE3">[1]&nbsp; Hu J, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Xia Q, Huang L, Zhang Y, Zhang C, Zeng Y, Liu H, Zhang S, Huang G,
Wan W, Ding Y, Hu F, Yang C, Chen L, Wang W. Identification of Key
Genes for the Ultrahigh Yield of Rice Using Dynamic Cross-tissue
Network Analysis. Genomics Proteomics Bioinformatics. 2020 Jul
28:S1672-0229(20)30091-7. doi: 10.1016/j.gpb.2019.11.007.
(Co-first author) </span></p>

      <p class="STYLE10"><span class="STYLE3">[2]&nbsp;
Yu X, Wang Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
. Essential gene expression pattern of head and neck squamous cell
carcinoma revealed by tumor-specific expression rule based on
single-cell RNA sequencing. Biochim Biophys Acta Mol Basis Dis. 2020
Mar 28;1866(7):165791. doi: 10.1016/j.bbadis.2020.165791.
(Co-corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[3]&nbsp;
Tang H, Tang Y, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L. Gene expression analysis reveals the tipping points during
infant brain development for human and chimpanzee. BMC Genomics. 2020
Mar 5;21(Suppl 1):74. doi: 10.1186/s12864-020-6465-8. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[4]&nbsp;
      </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Huang T, Lu C. Editorial: Machine Learning Advanced Dynamic Omics Data
Analysis for Precision Medicine. Front Genet. 2020 Feb 4;10:1343. doi:
10.3389/fgene.2019.01343. eCollection 2019. (Corresponding author) </span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[5]&nbsp;
Zhang S, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Hu B, Zhang YH, Feng K, Chen L, Niu Z, Li J, Huang T, Cai YD.
Discriminating Origin Tissues of Tumor Cell Lines by Methylation
Signatures and Dys-Methylated Rules. Front Bioeng Biotechnol. 2020 May
26;8:507. doi: 10.3389/fbioe.2020.00507. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[6]&nbsp;
Zhang S, Zhao J, Lv X, Fan J, Lu Y, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Wu H, Chen L, Zhao Y.
Analysis on gene modular network reveals morphogen-directed development
robustness in Drosophila.Cell Discov. 2020 Jun 30;6:43. doi:
10.1038/s41421-020-0173-z.
eCollection 2020. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[7]&nbsp;
Li Y, Omori A, Flores RL, Satterfield S, Nguyen C, Ota T, Tsurugaya T,
Ikuta T, Ikeo K, Kikuchi M, Leong JCK, Reich A, Hao M, Wan W, Dong Y,
Ren Y, Zhang S, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Uesaka M, Uchida Y, Li X, Shibata TF, Bino T,
Ogawa K, Shigenobu S, Kondo M, Wang F, Chen L, Wessel G, Saiga H,
Cameron RA, Livingston B, Bradham C, Wang W, Irie N.Genomic insights of
body plan transitions from bilateral to pentameral symmetry in
Echinoderms. Commun Biol. 2020 Jul 10;3(1):371. doi:
10.1038/s42003-020-1091-1. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[8]&nbsp;
Li ML, Tang H, Shao Y, Wang MS, Xu HB, Wang S, Irwin DM, Adeola AC, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Chen L, Li Y, Wu DD. Evolution and transition of expression trajectory
during human brain development. BMC Evol Biol. 2020 Jun 23;20(1):72.
doi: 10.1186/s12862-020-01633-4. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[9]&nbsp;
Yuan F, Pan X, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Zhang YH, Chen L, Gan Z, Huang T, Cai YD. Identifying Cell-Type
Specific Genes and Expression Rules Based on Single-Cell Transcriptomic
Atlas Data. Front Bioeng Biotechnol. 2020 Apr 29;8:350. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[10]&nbsp;
Li M, Pan X, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Feng K, Chen L, Huang T, Cai YD. Alternative
Polyadenylation Modification Patterns Reveal Essential
Posttranscription Regulatory Mechanisms of Tumorigenesis in Multiple
Tumor Types. Biomed Res Int. 2020 Jun 15;2020:6384120. doi:
10.1155/2020/6384120. eCollection 2020.</span> </p>

      <p class="STYLE10"><span class="STYLE3">[11]&nbsp;
Zhang Y, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Chen L, Ding S, Huang T, Cai YD. Identification of COVID-19
Infection-Related Human Genes Based on a Random Walk Model in a
Virus-Human Protein Interaction Network. Biomed Res Int. 2020 Jul
8;2020:4256301. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[12]&nbsp;
Pan X, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Chen L, Feng K, Huang T, Cai YD. Investigation
and Prediction of Human Interactome Based on Quantitative Features.
Front Bioeng Biotechnol. 2020 Jul 17;8:730. doi:
10.3389/fbioe.2020.00730. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[13]&nbsp;
Zhang YH, Pan XY, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L, Huang T, Cai YD. Identifying the RNA signatures of coronary
artery disease from combined lncRNA and mRNA expression profiles.
Genomics. 2020
Sep 9;S0888-7543(20)30508-5. doi: 10.1016/j.ygeno.2020.09.016. </span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[14]&nbsp;
Pan XY, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Chen L, Feng KY, Huang T, Cai YD. Investigation and
Prediction of Human Interactome Based on Quantitative Features. Front
Bioeng Biotechnol. 2020 Jul 17;8:730. doi: 10.3389/fbioe.2020.00730.</span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[15]&nbsp;
Chen L, Li Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Liu D, Li H, Huang T, Cai YD. Identifying Robust Microbiota
Signatures and Interpretable Rules to Distinguish Cancer Subtypes.
Front Mol Biosci. 2020 Nov 4;7:604794. doi: 10.3389/fmolb.2020.604794.</span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[16]&nbsp;
Zhang YH, Li Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Pan X, Chen L, Liu D, Li H, Huang T, Cai YD. Distinguishing
Glioblastoma Subtypes by Methylation Signatures. Front Genet. 2020 Nov
24;11:604336.</span> </p>

      <p class="STYLE10">-------------------------------------------------------2019-----------------------------------------------------<br>

      <span class="STYLE3">[1]&nbsp; Guo WF, Zhang SW,
      </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Akutsu T, Chen L. Network control principles for identifying
personalized driver genes in cancer. Brief Bioinform. 2019 Nov 10. pii:
bbz089. doi: 10.1093/bib/bbz089. (Co-corresponding author) </span></p>

      <p class="STYLE10"><span class="STYLE3">[2]&nbsp;
Guo W, Zhang SW, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Li Y, Gao J, Chen L. A novel network control model for identifying
personalized driver genes in cancer. PLoS Comput Biol. 2019 Nov
25;15(11):e1007520. doi: 10.1371/journal.pcbi.1007520. [Epub ahead of
print] (Co-corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[3]&nbsp;
      </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Dai H. Single-Cell RNA Sequencing-Based Computational Analysis to
Describe Disease Heterogeneity. Front Genet. 2019 Jul 12;10:629. doi:
10.3389/fgene.2019.00629. (Corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[4]&nbsp;
Lu Y, Fang Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Li M, Chen Q, Zhang H, Zhou Q, Hu Y, Chen L, Su S. Chronic hepatitis
B: dynamic change in Traditional Chinese Medicine syndrome by dynamic
network biomarkers. Chin Med. 2019 Nov 21;14:52. doi:
10.1186/s13020-019-0275-4. eCollection 2019. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[5]&nbsp;
Dai H, Li L, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Chen L. Cell-specific network constructed by single-cell RNA sequencing
data. Nucleic Acids Res. 2019 Mar 13. pii: gkz172. doi:
10.1093/nar/gkz172. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[6]&nbsp;
Tang H, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L. High-Order Correlation Integration for Single-Cell or Bulk
RNA-seq Data
Analysis.
Front Genet. 2019 Apr 26;10:371. doi: 10.3389/fgene.2019.00371.
(Co-corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[7]&nbsp;
Gao W, Sun YB,
Zhou WW, Xiong ZJ, Chen L, Li H, Fu TT, Xu K, Xu W, Ma L, Chen YJ,
Xiang XY, Zhou L, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Zhang S, Jin JQ, Chen HM, Zhang G, Hillis DM, Ji X, Zhang YP, Che J.
Genomic and transcriptomic investigations of the evolutionary
transition from oviparity to viviparity. Proc Natl Acad Sci U S A. 2019
Feb 26;116(9):3646-3655. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[8]&nbsp;
Lu Y, Fang Z, Li M, Qian C, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Lu L, Chen Q, Zhang H, Zhou Q, Sun Y, Xue X, Hu Y, Chen L, Su S.
Dynamic edge-based biomarker noninvasively predicts hepatocellular
carcinoma with hepatitis B virus infection for individual patients
based on blood testing.J Mol Cell Biol. 2019 Mar 29. pii: mjz025. doi:
10.1093/jmcb/mjz025. [Epub ahead of print] </span> </p>

      <p class="STYLE10"><span class="STYLE3">[9]&nbsp;
Gao D, Zhang L, Song D, Lv J, Wang L, Zhou S, Li Y, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zeng Y,
Zhang J, Wang X. Values of integration between lipidomics and clinical
phenomes in patients with acute lung infection, pulmonary embolism, or
acute exacerbation of chronic pulmonary diseases: a preliminary study.
J Transl Med. 2019 May 20;17(1):162. doi: 10.1186/s12967-019-1898-z. </span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[10]&nbsp;
Shi QQ, Hu B, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">and
Zhang CC. Multi-view Subspace Clustering Analysis for Aggregating
Multiple Heterogeneous Omics Data. Front. Genet., 20 August 2019 |
https://doi.org/10.3389/fgene.2019.00744 </span> </p>

      <p class="STYLE10"><span class="STYLE3">[11]&nbsp;
      </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Xiangtian Yu, Chengming Zhang.&#8197;Pan-cancer study of progressive
protein signatures for digestive cancers.Gut, 2019, 68 (Suppl 1)
A15-A16; DOI: 10.1136/gutjnl-2019-IDDFAbstracts.29. (ePoster)
(Corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[12]&nbsp;
Chen L, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Pan X, Zhang YH, Huang T and Cai YD. Identifying
Methylation Pattern and Genes Associated with Breast Cancer Subtypes.
Int. J. Mol. Sci. 2019, 20(17), 4269;
https://doi.org/10.3390/ijms20174269.
(Co-first
author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[13]&nbsp;
Song D, Tang L, Wang L, Huang J, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Fang H, Wang X. Roles of TGF&beta;1 in the expression of
phosphoinositide 3-kinase isoform genes and sensitivity and response of
lung telocytes to PI3K inhibitors. Cell Biol Toxicol. 2019 Sep 15. doi:
10.1007/s10565-019-09487-3. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[14]&nbsp;
Chen L, Pan XY, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Huang T, Cai YD. Identifying essential signature genes and
expression rules associated with distinctive development stages of
early embryonic cells. IEEE Access, 2019, 7:128570 - 128578.
DOI:10.1109/ACCESS.2019.2939556.
(Co-first
author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[15]&nbsp;
Pan XY, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Yuan F, Zhang YH, Chen L, Zhu LC, Wan SB, Huang T, Cai YD. Screening of
Methylation Signature and Gene Functions Associated With the Subtypes
of Isocitrate Dehydrogenase-Mutation Gliomas. Frontiers in
Bioengineering and Biotechnology, 2019, 7:339.
DOI=10.3389/fbioe.2019.00339. (Co-first
author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[16]&nbsp;
Chen L, Pan X, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang Y-H, Zhang Y, Huang T and Cai Y-D. Immunosignature Screening
for Multiple Cancer Subtypes Based on Expression Rule. Front. Bioeng.
Biotechnol. 2019, 7:370. doi: 10.3389/fbioe.2019.00370 (Co-first
author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[17]&nbsp;
Choobdar S, Ahsen ME, Crawford J, ..., </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, ..., Sven Bergmann &amp; Daniel Marbach. Assessment of network
module identification across complex diseases. Nature Methods,
16:843&ndash;852 (2019). (Consortium author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[18]&nbsp;
Zhang S, Pan X, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Guo W, Gan Z, Zhang YH, Chen L, Zhang Y, Huang T, Cai YD. Copy Number
Variation Pattern for Discriminating MACROD2 States of Colorectal
Cancer Subtypes. Front Bioeng Biotechnol. 2019 Dec 19;7:407. doi:
10.3389/fbioe.2019.00407. eCollection 2019.
(Co-first
author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[19]&nbsp;
Sun S, Yu X, Sun F, Tang Y, Zhao J, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
. Dynamically characterizing individual clinical change by the steady
state of disease-associated pathway. BMC Bioinformatics. 2019 Dec
24;20(Suppl 25):697. doi: 10.1186/s12859-019-3271-x. (Co-corresponding
author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[20]&nbsp;
Song D, Tang L, Huang J, Wang L, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Wang X. Roles of transforming growth factor-&beta; and
phosphatidylinositol 3-kinase isoforms in integrin &beta;1-mediated
bio-behaviors of mouse lung telocytes. J Transl Med. 2019 Dec
30;17(1):431. doi: 10.1186/s12967-019-02181-2. </span> </p>

      <p class="STYLE10">-------------------------------------------------------2018-----------------------------------------------------<br>

      <span class="STYLE3">[1]&nbsp; Wang L, Yu X,
Zhang C, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
. Detecting Personalized Determinants During Drug Treatment from Omics
Big Data. Curr Pharm Des. 2018;24(32):3727-3738. doi:
10.2174/1381612824666181106102111.
(Corresponding author) </span></p>

      <p class="STYLE10"><span class="STYLE3">[2]&nbsp;
      </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">, Lu Wang, Xiangtian Yu
. A revisit on personalized critical variance of gut microbiota during
the occurrence of Type I Diabetes. Gut, 2018, 67(Suppl 2):A7.
(ePoster) (Corresponding author) </span> </p>

      <p class="STYLE10"> <span class="STYLE3">[3]&nbsp;
Shaoyan Sun,
Xiangtian Yu,
Fengnan Sun, Ying Tang, Juan Zhao, and </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
. Characterizing and Discriminating Individual Steady State of
Disease-Associated Pathway. Intelligent Computing Theories and
Application, LNCS 10954, 2018, pp.533-538. (Co-corresponding author) </span>
      </p>

      <p class="STYLE10"> <span class="STYLE3">[4]&nbsp;
Hu
J, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Xia Q, Qian Q, Yang C, Ding Y, Chen L, Wang W.
Unravelling miRNA regulation in yield of rice (Oryza sativa) based on
differential network model. Scientific Reportsvolume 8, Article number:
8498, 2018. (Co-first
author) </span> </p>

      <p class="STYLE10"> <span class="STYLE3">[5]&nbsp;
Guo
Weifeng,
Zhang SW, Liu LL, Liu F, Shi QQ, Zhang L, Tang Y, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Chen L. Discovering personalized driver mutation profiles of single
samples in cancer by network control strategy. Bioinformatics. 2018 Jan
10. doi: 10.1093/bioinformatics/bty006. [Epub ahead of print]
(Co-corresponding author)</span> <br>

      </p>

      <p class="STYLE10"> <span class="STYLE3">[6]&nbsp;Guo
Weifeng, Zhang SW, Shi QQ,
Zhang CM, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Chen L. A novel algorithm for finding optimal driver nodes to target
control complex networks and its applications for drug targets
identification.&nbsp;&nbsp;&nbsp; BMC Genomics. 2018 Jan
19;19(Suppl
1):924. doi: 10.1186/s12864-017-4332-z. (Co-corresponding author) </span><br>

      </p>

      <p class="STYLE10"> <span class="STYLE3">[7]&nbsp;
Zhang JS, Guo JM, Zhang M, Yu XT, Yu XQ, Guo WF, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">, Chen LN. Efficient Mining Multi-mers in a
Variety of Biological Sequences.
doi:10.1109/TCBB.2018.2828313. (Co-corresponding author) </span><br>

      </p>

      <p class="STYLE10"> <span class="STYLE3">[8]&nbsp;
Yu XT, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">. Integrative Analysis of Omics Big Data.
Methods Mol Biol. 2018;1754:109-135. doi: 10.1007/978-1-4939-7717-8_7.
(Chapter) </span><br>

      </p>

      <p class="STYLE10"> <span class="STYLE3">[9]&nbsp;
Yu XT, Wang L, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">. Revisit of Machine Learning Supported
Biological and Biomedical Studies.
Methods Mol Biol. 2018;1754:183-204. doi: 10.1007/978-1-4939-7717-8_11.
(Chapter) </span><br>

      </p>

      <p class="STYLE10"> <br>

-------------------------------------------------------2017-----------------------------------------------------<br>

      <span class="STYLE12">[1] Yu X, Zhang J, Sun S, Zhou
X, <strong>Zeng T</strong>, Chen L. Individual-specific
edge-network analysis for disease prediction. Nucleic Acids Res. 2017
Nov 16;45(20):e170. doi: 10.1093/nar/gkx787. PubMed
PMID: 28981699. (Co-corresponding author) </span></p>

      <p class="STYLE3">[2] Shi Q, Zhang C, Peng M, Yu X, <strong>Zeng
T</strong>, Liu J, Chen L. Pattern fusion analysis by adaptive
alignment of multiple heterogeneous omics data. Bioinformatics. 2017
Sep 1;33(17):2706-2714. doi: 10.1093/bioinformatics/btx176. PubMed
PMID: 28520848. (Co-corresponding author)</p>

      <p class="STYLE3">[3] Zhang C, Liu J, Shi Q, <strong>Zeng
T</strong>
, Chen L. Comparative network stratification analysis for identifying
functional interpretable network biomarkers. BMC Bioinformatics. 2017
Mar 14;18(Suppl 3):48. doi: 10.1186/s12859-017-1462-x. PubMed
PMID: 28361683. (Co-corresponding author) </p>

      <p class="STYLE3">[4] Li M, Li C, Liu WX, Liu C, Cui
J, Li Q, Ni H, Yang Y, Wu C, Chen C, Zhen X, <strong>Zeng
T</strong>
, Zhao M, Chen L, Wu J, Zeng R, Chen L. Dysfunction of PLA2G6 and
CYP2C44-associated network signals imminent carcinogenesis from chronic
inflammation to hepatocellular carcinoma. J Mol Cell Biol. 2017 Jun 26.
doi: 10.1093/jmcb/mjx021. [Epub ahead of print] PubMed
PMID: 28655161. </p>

      <p class="STYLE3">[5] Shi Q, Zhang C, Guo W, <strong>Zeng
T</strong>
, Lu L, Jiang Z, Wang Z, Liu J, Chen L. Local network component
analysis for quantifying transcription factor activities. Methods. 2017
Jul 15;124:25-35. doi: 10.1016/j.ymeth.2017.06.018. Epub 2017 Jul 12.
PubMed
PMID: 28710010. )</p>

      <p class="STYLE3">[7] Zhang C, Liu J, Shi QQ, <strong>Zeng
T</strong>
, Chen L. Differential function analysis: identifying structure
and activation variations in dysregulated pathways. SCIENCE CHINA
Information Sciences, Vol. 60 012108:1&ndash;012108:16, 2017.
(Co-corresponding author) </p>

      <p class="STYLE10">-------------------------------------------------------2016-----------------------------------------------------<br>

      <span class="STYLE12">[1] Yang Y, Lin X, Lu X, Luo
G, <strong>Zeng T</strong>, Tang J, Jiang F, Li L, Cui X,
Huang W, Hou G, Chen X, Ouyang Q, Tang S, Sun H, Chen L, Gonzalez FJ,
Wu M, Cong W, Chen L, Wang H. Interferon-microRNA signalling drives
liver precancerous lesion formation and hepatocarcinogenesis. Gut. 2016
Feb 9. pii: gutjnl-2015-310318. doi: 10.1136/gutjnl-2015-310318. [Epub
ahead of print] PubMed PMID: 26860770.</span></p>

      <p class="STYLE10">-------------------------------------------------------2015-----------------------------------------------------<br>

      <span class="STYLE12">[1] <strong>Zeng T</strong>,
Zhang W, Yu X, Liu X, Li M, Chen L. Big-data-based edge biomarkers:
study on dynamical drug sensitivity and resistance in individuals.
Brief Bioinform. 2015 Sep 26. pii: bbv078. [Epub ahead of print] PubMed
PMID: 26411472.</span></p>

      <p class="STYLE3">[2] Yu X, <strong>Zeng T</strong>,
Li G. Integrative enrichment analysis: a new computational method to
detect dysregulated pathways in heterogeneous samples. BMC Genomics.
2015 Nov 10;16:918. doi: 10.1186/s12864-015-2188-7. PubMed PMID:
26556243; PubMed Central PMCID: PMC4641376. (Co-corresponding author)</p>

      <p class="STYLE3">[3] Kang H, Cho KH, Zhang XD, <strong>Zeng
T</strong>, Chen L. Inferring Sequential Order of Somatic
Mutations during Tumorgenesis based on Markov Chain Model. IEEE/ACM
Trans Comput Biol Bioinform. 2015 Sep-Oct;12(5):1094-103. doi:
10.1109/TCBB.2015.2424408. PubMed PMID: 26451822. (Co-corresponding
author)</p>

      <p class="STYLE3">[4] Yu X, <strong>Zeng T</strong>,
Wang X, Li G, Chen L. Unravelling personalized dysfunctional gene
network of complex diseases based on differential network model. J
Transl Med. 2015 Jun 13;13:189. doi: 10.1186/s12967-015-0546-5. PubMed
PMID: 26070628; PubMed Central PMCID: PMC4467679.</p>

      <p class="STYLE3">[5] Shi Q, Liu X, <strong>Zeng
T</strong>, Wang W, Chen L. Detecting disease genes of non-small
lung cancer based on consistently differential interactions. Cancer
Metastasis Rev. 2015 Jun;34(2):195-208. doi: 10.1007/s10555-015-9561-5.
PubMed PMID: 26004969.</p>

      <p class="STYLE3">[6] Yang D, Zhou J,<strong>
Zeng T</strong>, Yang Z, Wang X, Hu J, Song Y, Chen L, Peer D,
Wang X, Bai C. Serum chemokine network correlates with chemotherapy in
non-small cell lung cancer. Cancer Lett. 2015 Aug 28;365(1):57-67. doi:
10.1016/j.canlet.2015.05.001. Epub 2015 May 11. PubMed PMID: 25976768.</p>

      <p class="STYLE3">[7] Chen S, Liu J, <strong>Zeng
T</strong>. Measuring the quality of linear patterns in
biclusters. Methods. 2015 Jul 15;83:18-27. doi:
10.1016/j.ymeth.2015.04.005. Epub 2015 Apr 15. PubMed PMID: 25890245.</p>

      <p class="STYLE3">[8] Zhang W, <strong>Zeng T</strong>,
Liu X, Chen L. Diagnosing phenotypes of single-sample individuals by
edge biomarkers. J Mol Cell Biol. 2015 Jun;7(3):231-41. doi:
10.1093/jmcb/mjv025. Epub 2015 Apr 26. PubMed PMID: 25917597. (Co-first
author)</p>

      <p class="STYLE10">-------------------------------------------------------2014-----------------------------------------------------<br>

      <span class="STYLE12">[1] <strong>Zeng T</strong>,
Zhang W, Yu X, Liu X, Li M, Liu R, Chen L. Edge biomarkers for
classification and prediction of phenotypes. Sci China Life Sci. 2014
Nov;57(11):1103-14. doi: 10.1007/s11427-014-4757-4. Epub 2014 Oct 17.
Review. PubMed PMID: 25326072.</span></p>

      <p class="STYLE3">[2] Wen Z, Zhang W,<strong>
Zeng T</strong>, Chen L. MCentridFS: a tool for identifying
module biomarkers for multi-phenotypes from high-throughput data. Mol
Biosyst. 2014 Nov;10(11):2870-5. doi: 10.1039/c4mb00325j. PubMed PMID:
25099602.</p>

      <p class="STYLE3">[3] <strong>Zeng T</strong>,
Wang DC, Wang X, Xu F, Chen L. Prediction of dynamical drug sensitivity
and resistance by module network rewiring-analysis based on
transcriptional profiling. Drug Resist Updat. 2014 Jul;17(3):64-76.
doi: 10.1016/j.drup.2014.08.002. Epub 2014 Aug 13. Review. PubMed PMID:
25156319.</p>

      <p class="STYLE3">[4] Zhang W, <strong>Zeng T</strong>,
Chen L. EdgeMarker: Identifying differentially correlated molecule
pairs as edge-biomarkers. J Theor Biol. 2014 Dec 7;362:35-43. doi:
10.1016/j.jtbi.2014.05.041. Epub 2014 Jun 12. PubMed PMID: 24931676.
(Co-corresponding author)</p>

      <p class="STYLE3">[5] <strong>Zeng T</strong>,
Zhang CC, Zhang W, Liu R, Liu J, Chen L. Deciphering early development
of complex diseases by progressive module network. Methods. 2014 Jun
1;67(3):334-43. doi: 10.1016/j.ymeth.2014.01.021. Epub 2014 Feb 21.
PubMed PMID: 24561825.</p>

      <p class="STYLE3">[6] Zhao M, Chang HT, Zhou Q, <strong>Zeng
T</strong>, Shih CS, Liu ZP, Chen L, Wei DQ. Predicting
protein-ligand interactions based on chemical preference features with
its application to new D-amino acid oxidase inhibitor discovery. Curr
Pharm Des. 2014;20(32):5202-11. PubMed PMID: 24410568.</p>

      <p class="STYLE3">[7] Li M, <strong>Zeng T</strong>,
Liu R, Chen L. Detecting tissue-specific early warning signals for
complex diseases based on dynamical network biomarkers: study of type 2
diabetes by cross-tissue analysis. Brief Bioinform. 2014
Mar;15(2):229-43. doi: 10.1093/bib/bbt027. Epub 2013 Apr 25. PubMed
PMID: 23620135. (Co-first author)</p>

      <p class="STYLE10">------------------------------------------------2013
and before-------------------------------------------------<br>

      <span class="STYLE12">[1]<strong> Zeng T</strong>,
Sun SY, Wang Y, Zhu H, Chen L. Network biomarkers reveal dysfunctional
gene regulations during disease progression. FEBS J. 2013
Nov;280(22):5682-95. doi: 10.1111/febs.12536. Epub 2013 Oct 22. Review.
PubMed PMID: 24107168.</span></p>

      <p class="STYLE3">[2] Tarca AL, Lauria M, Unger M,
Bilal E, Boue
S, Kumar Dey K, Hoeng J, Koeppl H, Martin F, Meyer P, Nandy P, Norel R,
Peitsch M, Rice JJ, Romero R, Stolovitzky G, Talikka M, Xiang Y,
Zechner C; IMPROVER DSC Collaborators. Strengths and limitations of
microarray-based phenotype prediction: lessons learned from the
IMPROVER Diagnostic Signature Challenge. Bioinformatics. 2013 Nov
15;29(22):2892-9. doi: 10.1093/bioinformatics/btt492. Epub 2013 Aug 20.
(Collaborators) </p>

      <p class="STYLE3">[3] Sun SY, Liu ZP, <strong>Zeng
T</strong>, Wang Y, Chen L. Spatio-temporal analysis of type 2
diabetes mellitus based on differential expression networks. Sci Rep.
2013;3:2268. doi: 10.1038/srep02268. PubMed PMID: 23881262; PubMed
Central PMCID: PMC3721080.</p>

      <p class="STYLE3">[4] <strong>Zeng T</strong>,
Chen L. Tracing dynamic biological processes during phase transition.
BMC Syst Biol. 2012;6 Suppl 1:S12. doi: 10.1186/1752-0509-6-S1-S12.
Epub 2012 Jul 16. Erratum in: BMC Syst Biol. 2012 Jul 16;6 Suppl 1:S23.
PubMed PMID: 23046764; PubMed Central PMCID: PMC3403121.</p>

      <p class="STYLE3">[5] Xiong Y, Liu J, Zhang W, <strong>Zeng
T</strong>. Prediction of heme binding residues from protein
sequences with integrative sequence profiles. Proteome Sci. 2012 Jun
21;10 Suppl 1:S20. doi: 10.1186/1477-5956-10-S1-S20. PubMed PMID:
22759579; PubMed Central PMCID: PMC3380730.</p>

      <p class="STYLE3">[6]<strong> Zeng T</strong>,
Li J, Liu J. Distinct interfacial biclique patterns between
ssDNA-binding proteins and those with dsDNAs. Proteins. 2011
Feb;79(2):598-610. doi: 10.1002/prot.22908. PubMed PMID: 21120860.</p>

      <p class="STYLE3">[7]<strong> Zeng T</strong>,
Li J. Maximization of negative correlations in time-course gene
expression data for enhancing understanding of molecular pathways.
Nucleic Acids Res. 2010 Jan;38(1):e1. doi: 10.1093/nar/gkp822. Epub
2009 Oct 23. PubMed PMID: 19854949; PubMed Central PMCID: PMC2800212.</p>

      <p class="STYLE3">[8] <strong>Zeng T</strong>,
Liu J. Mixture classification model based on clinical markers for
breast cancer prognosis. Artif Intell Med. 2010 Feb-Mar;48(2-3):129-37.
doi: 10.1016/j.artmed.2009.07.008. Epub 2009 Dec 14. PubMed PMID:
20005686.</p>

      <p class="STYLE3">[9] Gu C, <strong>Zeng T</strong>,
Li Y, Xu Z, Mo Z, Zheng C. Structure-function analysis of mutant
RNA-dependent RNA polymerase complexes with VPg. Biochemistry (Mosc).
2009 Oct;74(10):1132-41. PubMed PMID: 19916926. (Co-first author) </p>

      <p class="STYLE3">[10] <strong>Zeng T</strong>,
Zhongxi Mo. Bioinformatics analysis of mutant structure and functional
sites of foot-and-mouth disease virus RNA-dependent RNA polymerase.
Journal of Computational and Theoretical Nanoscience, v 4, n 7-8, p
1330-1336, November/December 2007. </p>

      <p class="STYLE3">[11] Zhongxi Mo, <strong>Zeng
T</strong> . Improved genetic algorithm for problem of genome
rearrangement. Wuhan University Journal of Natural Sciences, v 11, n 3,
p 498-502, May 2006.<br>

      </p>

      <p class="STYLE3">&nbsp;</p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><em>Conference paper:</em><br>

      <span class="STYLE12">[1] Lu wang, Xiangtian Yu, <strong>Tao
Zeng</strong>.
A revisit on personalized critical variance of gut microbiota during
the occurrence of Type I Diabetes. IDDF 2018. (ePoster) </span></p>

      <p class="STYLE3">[2] <strong>Tao Zeng</strong>.
Improving the
discrimination and interpretability for complex diseases by network
biomarker. 16th SCBA International Symposium, 2017, pp.298. (Abstract) </p>

      <p class="STYLE3"> [3] Weifeng Guo, Shaowu Zhang,
Qianqian Shi, Chengming Zhang, <strong>Tao Zeng</strong>
and Luonan Chen . A novel algorithm for finding optimal driver nodes to
target control complex networks and its applications for drug targets
identification. International Conference on Bioinformatics (InCoB)
2017. </p>

      <p class="STYLE3">[4] Chuanchao Zhang, Juan Liu,
Qianqian Shi, <strong>Tao Zeng</strong> and Luonan Chen.
Comparative network stratification analysis for identifying functional
interpretable network biomarkers. The Asia Pacific Bioinformatics
Conference (APBC) 2017. </p>

      <p class="STYLE3"> [5] Chuanchao Zhang, Juan Liu, <strong>Tao
Zeng</strong>, and Luonan Chen. Integration of multiple
heterogeneous omics data
.The IEEE International Conference on Bioinformatics and Biomedicine
(BIBM) 2016. </p>

      <p class="STYLE3">[6] Chuanchao Zhang, Juan Liu,
Qianqian Shi, <strong>Tao Zeng</strong>, Luonan Chen.
Identification of phenotypic networks based on whole transcriptome by
comparative network decomposition. 2015 IEEE International Conference
on Bioinformatics and Biomedicine (BIBM), pp: 189-194. </p>

      <p class="STYLE3">[7] <strong>Tao Zeng</strong>,
Ziming Wang, Luonan Chen: Detecting Phosphorylation Determined Active
Protein Interaction Network during Cancer Development by Robust Network
Component Analysis. DTMBIO '14 Proceedings of the ACM 8th International
Workshop on Data and Text Mining in Bioinformatics ,DTMBIO@CIKM 2014,
pp.49. (Abstract)</p>

      <p class="STYLE3">[8] <strong>Tao Zeng</strong>,
Ruozhao Wang, Luonan Chen. Module network rewiring in response to
therapy. IEEE 6th International Conference on Systems Biology, ISB
2012, p 197-202.</p>

      <p class="STYLE3">[9] <strong>Tao Zeng</strong>,
Luonan Chen. Identifying temporal trace of biological process during
phase transition. Systems Biology (ISB), 2011 IEEE International
Conference on, pp:368-373.</p>

      <p class="STYLE3">[10] Guanying Piao, Bangguo Qian,
Shigeru Saito, ZP Liu, <strong>Tao Zeng</strong>, Y. Wang,
Jiarui Wu, Huarong Zhou, L. Chen, K. Horimoto. Phenotype-difference
oriented identification of molecular functions for diabetes progression
in Goto-Kakizaki rat. Systems Biology (ISB), 2011 IEEE International
Conference on, pp:111-116.</p>

      <p class="STYLE3">[11] Yi Xiong, Wen Zhang, <strong>Tao
Zeng</strong>, Juan Liu. Prediction of heme binding sites in heme
proteins using an integrative sequence profile coupling evolutionary
information with physicochemical properties. Proceedings - 2011 IEEE
International Conference on Bioinformatics and Biomedicine, BIBM 2011,
p 143-146, 2011.</p>

      <p class="STYLE3">[12] Jinyan Li, Qian Liu, <strong>Tao
Zeng</strong>. Negative correlations in collaboration: concepts
and algorithms. Knowledge Discovery and Data Mining, KDD2010, p
463-472, 2010.</p>

      <p class="STYLE3">[13] <strong>Tao Zeng</strong>,
Xuan Guo, Juan Liu. Discovering negative correlated gene sets from
integrative gene expression data for cancer prognosis. Proceedings -
2010 IEEE International Conference on Bioinformatics and Biomedicine,
BIBM 2010, p 489-492, 2010, </p>

      <p class="STYLE3">[14] <strong>Tao Zeng</strong>,
Fei Luo, Juan Liu. Biological pathway conducting microarray-based
cancer classification. Proceedings of the 2009 2nd International
Conference on Biomedical Engineering and Informatics, BMEI 2009, 2009.</p>

      <p class="STYLE3">[15] <strong>Tao Zeng</strong>,
Juan Liu. A novel mixture classifier and its application in breast
cancer prognosis. 2nd International Conference on Bioinformatics and
Biomedical Engineering, iCBBE 2008, p 564-568, 2008.</p>

      <p class="STYLE3">[16] <strong>Tao Zeng</strong>,
Juan Liu. Analysis on time-lagged gene clusters in time series gene
expression data. Proceedings - 2007 International Conference on
Computational Intelligence and Security, CIS 2007, p 182-185, 2007.</p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><em>Chinese book &amp;
chapter:</em><br>

      <span class="STYLE12">[1] &#38472;&#27931;&#21335;, <strong>&#26366;&#28059;</strong>.
2.13 &#22823;&#25968;&#25454;&#26102;&#20195;&#30340;&#29983;&#29289;&#20449;&#24687;&#23398;&#19982;&#31995;&#32479;&#29983;&#29289;&#23398;&#26032;&#36827;&#23637;. &#20013;&#22269;&#31185;&#23398;&#38498;&#12298;2016&#31185;&#23398;&#21457;&#23637;&#25253;&#21578;&#12299;, ISBN&#65306;978-7-03-048895-4. </span></p>

      <span class="STYLE12">[2] &#21016;&#27835;&#24179;, &#36213;&#20852;&#26126;, <strong>&#26366;&#28059;</strong>,
&#38472;&#27931;&#21335;. &#31532;&#20843;&#31456;&#29983;&#29289;&#22823;&#20998;&#23376;&#20449;&#21495;&#20256;&#23548;&#21644;&#32593;&#32476;. &#20013;&#22269;&#23398;&#31185;&#21457;&#23637;&#25112;&#30053;-&#29702;&#35770;&#19982;&#35745;&#31639;&#21270;&#23398;, ISBN&#65306;9787030489197. </span>
      <p></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE8">Softwares:</p>

      <p class="STYLE4">&hearts;&nbsp;MCentridFS (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/MCentridFS.rar">Multi-class
Centroid Feature Selection</a>): supply a tool for the optimal
module analysis on the biological active network </p>

      <p class="STYLE4">&hearts;&nbsp;PMNminer (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/PMNminer.rar">Progressive
module network analysis</a>): supply a computational framework
for the cross-tissue analysis on the phenotype-dynamics of biological
process (e.g. disease occurance and progression) </p>

      <p class="STYLE4">&hearts;&nbsp;IEApackage (<a href="https://github.com/bluesky2009/integrative-enrichment-analysis">Integrative
enrichment analysis</a>): supply a software package for the
functional enrichment analysis on the heterogeneous samples </p>

      <p class="STYLE4">&hearts;&nbsp;PFA (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/PFApackage_0.1.rar">Pattern
fusion analysis</a>): supply an integrative tool to identify
sample-patterns from different omics profiles by optimally adjusting
the effects of each data type to the hidden patterns. </p>

      <p class="STYLE4">&hearts;&nbsp;CNS (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/CNSpackage_0.1.rar">Comparative
Network Stratification</a>): supply a computational methods to
directly identify functional network biomarkers with both
discriminative power on disease states and readable interpretation on
biological functions. </p>

      <p class="STYLE4">&hearts;&nbsp;DFApackage (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/DFApackage.zip">Differential
function analysis</a>): supply a general framework to identify
the changes of element-structure and expression-activation of
biological functions, based on comparative non-negative matrix
factorization (cNMF). </p>

      <p class="STYLE4">&hearts;&nbsp;SCS (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/supplemntary_MATERIAL.zip">Single-sample
Controller Strategy</a>): supply a new single-sample controller
strategy (SCS) to identify personalized driver mutation profiles from
network controllability perspective. </p>

      <p class="STYLE4">&hearts;&nbsp;eTMpy (<a href="https://github.com/ztpub/eTMpy">Endogenous target
mimic (eTMs) screen package</a>): supply a python script for
screening the endogenous target mimic in plant. </p>

      <p class="STYLE3">&nbsp;</p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE1"><span class="STYLE9"><span class="STYLE8">Photos</span>:</span><br>

      </span></p>

      <span class="STYLE4"><a href="http://taulab.cn/images/2019-1.jpg">Mid-term report
meeting
(2019)</a></span> <span class="STYLE4"><a href="http://taulab.cn/images/2019-2.jpg">CAS Life Science
Forum (2019)</a></span> <span class="STYLE4"><a href="http://taulab.cn/images/CBC2019.jpg">The Fourth CCF
Bioinformatics Conference (2019)</a></span> <span class="STYLE4"><a href="http://taulab.cn/images/ChinaSBC-shanghai2019.JPG">The
first Chinese Systems Biology Conference (2019)</a></span>
      <p></p>

      <p></p>

      </td>

    </tr>

  </tbody>
</table>

<br>

<!-- bottom menu table -->
<!-- bottom menu table --><font face="Verdana" size="1">&copy;2018 Homepage of Tao Zeng <b>
&tau;&trade;</b></font><br>

</div>

<span style="display: none;"><a href="http://www.mianfeimoban.com">mianfeimoban</a></span>
<script language="JavaScript">
<!--
var caution = false
function setCookie(name, value, expires, path, domain, secure){
var curCookie = name + "=" + escape(value) +
((expires) ? "; expires=" + expires.toGMTString() : "") +
((path) ? "; path=" + path : "") +
((domain) ? "; domain=" + domain : "") +
((secure) ? "; secure" : "")
if (!caution || (name + "=" + escape(value)).length<= 4000)
document.cookie = curCookie
else
if (confirm("Cookie exceeds 4KB and will be cut!"))
document.cookie = curCookie
}
function getCookie(name) {
var prefix = name + "="
var cookieStartIndex = document.cookie.indexOf(prefix)
if (cookieStartIndex == -1)
return null
var cookieEndIndex = document.cookie.indexOf(";", cookieStartIndex+ prefix.length)
if (cookieEndIndex == -1)
cookieEndIndex = document.cookie.length
return unescape(document.cookie.substring(cookieStartIndex +prefix.length, cookieEndIndex))
}
function deleteCookie(name, path, domain) {
if (getCookie(name)) {
document.cookie = name + "=" +
((path) ? "; path=" + path : "") +
((domain) ? "; domain=" + domain : "") +
"; expires=Thu, 01-Jan-70 00:00:01 GMT"
}
}
function fixDate(date) {
var base = new Date(0)
var skew = base.getTime()
if (skew > 0)
date.setTime(date.getTime() - skew)
}
var now = new Date()
fixDate(now)
now.setTime(now.getTime() + 365 * 24 * 60 * 60 * 1000)
var visits = getCookie("counter")
if (!visits)
visits = 1
else
visits = parseInt(visits) + 1
setCookie("counter", visits, now)
document.write("There are " + visits + " visitions&#65281;")
// -->
</script>
<script type="text/javascript" src="http://tajs.qq.com/stats?sId=65927253" charset="UTF-8"></script>
=======
<script type="text/javascript" src="http://tajs.qq.com/stats?sId=65927253" charset="UTF-8"></script>
<div align="center">
<h1><font face="Verdana" size="7">Dr. Tao Zeng</font></h1>

<!-- top menu table -->
<!-- top menu table --><br>

<table cellpadding="5" cellspacing="5">

  <tbody>

    <tr>

      <td valign="top" width="25%"><!-- left box table -->
      <table bgcolor="#ffffff" cellpadding="0" cellspacing="1">

        <tbody>

          <tr>

            <td>
            <table bgcolor="#bbbbbb" width="100%">

              <tbody>

                <tr>

                  <td> <font face="Verdana" size="1">
                  <b>News and Photos</b></font> </td>

                </tr>

              </tbody>
            </table>

            </td>

          </tr>

          <tr>

            <td>
            <table bgcolor="#ffffff" width="100%">

              <tbody>

                <tr>

                  <td> <font face="Verdana" size="1">Dr.
Tao Zeng in FuDan University ...2016.5</font><br>

                  <div style="text-align: justify;"><font face="Verdana" size="1">taken an talk about "network
biomarker" in ShangHai Bioinformatics seminar</font></div>

                  <font face="Verdana" size="1"><br>

Dr.
Tao Zeng in BeiJing ................2016.4</font><br>

                  <div style="text-align: justify;"><font face="Verdana" size="1">served
as the academic secretary of The 557th Session of XIANGSHAN-SCIENCE
Conferences: Big Data, Precision Medicine and Bioinformatics</font><br>

                  </div>

                  <font face="Verdana" size="1"><br>

Dr.
Tao Zeng in ShangHai .............2016.3<br>

                  <br>

                  <img style="width: 243px; height: 333px;" src="images/zt.jpeg"><br>

                  <br>

                  </font><font face="Verdana" size="1"><br>

Dr.
Tao Zeng in Hong Kong.............2018.6<br>

                  <br>

                  <img style="width: 243px; height: 333px;" src="images/zt20180615.jpg"> <br>

                  <br>

                  </font><font face="Verdana" size="1"><br>

Dr.
Tao Zeng in ISB2018 (GuiYang) .............2018.8.20<br>

                  <br>

                  <img style="width: 243px; height: 183px;" src="images/ISB2018-zt.jpg"> <br>

                  </font><font face="Verdana" size="1"><br>

Dr.
Tao Zeng in The 9th National Conference on Bioinformatics and Systems
Biology of China.............2020.10<br>

                  <br>

                  <img style="width: 243px; height: 183px;" src="images/2020-10.jpg"> <br>

                  <br>

                  <br>

                  <br>

                  </font> </td>

                </tr>

              </tbody>
            </table>

            </td>

          </tr>

        </tbody>
      </table>

<!-- left box table --> </td>

      <td valign="top">
      <p class="STYLE10"><span class="STYLE8">Associate
Professor</span><br>

      <span class="STYLE3"> &nbsp;&nbsp;CAS Key
Laboratory of Computational Biology, Bio-Med Big Data Center, <br>

&nbsp;&nbsp; &nbsp;&nbsp; Shanghai
Institute of Nutrition and Health, <br>

&nbsp;&nbsp; &nbsp;&nbsp; &nbsp;&nbsp;
University of Chinese Academy of
Sciences, Chinese Academy of Sciences </span></p>

      <p class="STYLE3"> &nbsp;&nbsp;Present
address (office): Cell building 333, 320 YueYang Road, Shanghai 200031,
China<br>

&nbsp;&nbsp;Phone (office): +86-21-5492-0102<br>

&nbsp;&nbsp;E-mail: zengtao(at)bsbii.cn; zengtao(at)sibs.ac.cn<br>

&nbsp;&nbsp;Sites online: <a href="https://www.researchgate.net/profile/Tao_Zeng4">Researchgate</a>,
      <a href="http://www.researcherid.com/rid/A-1711-2012">Researcher
ID</a>, <a href="http://www.scholarmate.com/profile/taoceng36">ScholarMate</a>
, <a href="https://loop.frontiersin.org/people/552744/overview">Loop</a>
, <a href="https://www.linkedin.com/in/tazeng">LinkedIn</a>,
      <a href="https://orcid.org/0000-0002-0295-3994">ORCID</a>
, <a href="https://github.com/ztpub">GitHub</a></p>

      <p></p>

      <p></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE8">Education:</span><br>

      <span class="STYLE4">&bull;&nbsp;2006~2010 <a href="http://www.whu.edu.cn/">Wuhan University</a>
, <a href="http://cs.whu.edu.cn/">School of computer</a>,
Received the doctor's degree.<br>

&bull;&nbsp;2003~2006 <a href="http://www.whu.edu.cn/">Wuhan
University</a>, <a href="http://maths.whu.edu.cn/">School
of mathematics
and statistics </a>, Received the master's degree.<br>

&bull;&nbsp;1999~2003 <a href="http://www.whu.edu.cn/">Wuhan
University</a>, <a href="http://maths.whu.edu.cn/">School
of mathematics
and statistics </a>, Received the bachelor's degree.</span></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE8">Research
Interests:</span><br>

      <span class="STYLE4">&nbsp;&nbsp;Bioinformatics,
Computational systems biology,
Machine learning, Graph theory, Precision medicine</span></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE8">Research
and professional experence:</span><br>

      <span class="STYLE4">&bull;&nbsp;2019/7-now,
      <strong>associate Professor</strong>, <a href="http://www.bsbii.cn/">Shanghai Research Center for
Brain Science and Brain-Inspired Intelligence</a> <br>

      <span class="STYLE4">&bull;&nbsp;2014/12-2019/07,
      <strong>associate Professor</strong>, <a href="http://www.sysbio.ac.cn/">Key Laboratory
of Systems Biology </a>, <a href="http://www.sibcb.ac.cn/">Institute
of Biochemistry and Cell Biology </a>, <a href="http://www.sibs.cas.cn/">Shanghai
Institutes for Biological Sciences </a>, <a href="http://english.cas.cn/">Chinese Academy of
Sciences </a><br>

&bull;&nbsp;2013/01-2014/11, <strong>assistant Professor</strong>,
      <a href="http://www.sysbio.ac.cn/">Key Laboratory
of Systems Biology </a>, <a href="http://www.sibs.cas.cn/">Shanghai
Institutes for Biological Sciences </a>, <a href="http://english.cas.cn/">Chinese Academy of
Sciences </a><br>

&bull;&nbsp;2011/01-2012/12, <strong>post-doctor</strong>,
      <a href="http://www.sysbio.ac.cn/">Key Laboratory
of Systems Biology </a>, <a href="http://www.sibs.cas.cn/">Shanghai
Institutes for Biological Sciences </a>, <a href="http://english.cas.cn/">Chinese Academy of
Sciences </a><br>

      </span><span class="STYLE4">&bull;&nbsp;2008/07-2009/12,
      <strong>research-associate</strong>, <a href="http://www3.ntu.edu.sg/birc/">BioInformatics
Research Centre </a>, <a href="http://scse.ntu.edu.sg/Pages/Home.aspx">
School of Computer Engineering </a>, <a href="http://www.ntu.edu.sg/Pages/home.aspx">
Nanyang Technological
University </a>, Singapore</span></span></p>

      <p class="STYLE10"></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE8">Academic
activities:</span><br>

      <span class="STYLE4">&bull;&nbsp;The
academic secretary of <a href="http://www.xssc.ac.cn/xs/showconf.asp?pid=455&amp;tid=4">The
557th Session of XIANGSHAN-SCIENCE Conferences: Big Data, Precision
Medicine and Bioinformatics</a> <br>

&bull;&nbsp;The
organizing chair of <a href="http://sysbio.sibcb.ac.cn/cb/BioSynergy2015.html">Bio-synergy
2015</a> <br>

&bull;&nbsp;The
invited report at School of Mathematical Sciences, Peking University
(2020)<br>

&bull;&nbsp;The
invited report at The 9th National Conference on Bioinformatics and
Systems Biology of China, Shanghai (2020) <br>

&bull;&nbsp;The
invited report at 1st CAS Life Science Forum, Urumqi (2019) <br>

&bull;&nbsp;The
invited report at School of Mathematics, South China University of
Technology (2019)<br>

&bull;&nbsp;The
invited report at The Fourth CCF Bioinformatics Conference, GuangZhou
(CBC 2019)<br>

&bull;&nbsp;The
invited report at School of Mathematics, South China University of
Technology (2018)<br>

&bull;&nbsp;The
invited report at National Tianyuan Northwest Center for Mathematics,
Xi`an Jiaotong University (2018)<br>

&bull;&nbsp;The
Editor of <a href="https://www.frontiersin.org/research-topics/8239/machine-learning-advanced-dynamic-omics-data-analysis-for-precision-medicine">
Research Topics (2018) - Machine learning advanced dynamic omics data
analysis
for precision medicine </a> <br>

&bull;&nbsp;The
Editor of <a href="https://www.frontiersin.org/research-topics/12828/cross-domain-analysis-for-all-of-us-precision-medicine">
Research Topics (2020) - Cross-Domain Analysis for &ldquo;All of
Us&rdquo; Precision Medicine </a> <br>

&bull;&nbsp;The
Editor of <a href="https://www.frontiersin.org/research-topics/15661/rna-modification-and-non-coding-rnas-in-human-disease">
Research Topics (2020) - RNA Modification and Non-Coding RNAs in Human
Disease </a> <br>

&bull;&nbsp;The report on Highlights sessions of <a href="http://www.aporc.org/ISB/2015/">ISB 2015</a> and
      <a href="http://www.aporc.org/ISB/2018/">ISB 2018</a>
      <br>

&bull;&nbsp;The
invited report at Shanghai Bioinformatics Seminar (2016)<br>

&bull;&nbsp;The
invited report at School of Mathematics, South China University of
Technology (2016)<br>

&bull;&nbsp;The invited talk on <a href="https://dl.ccf.org.cn/meeting/loadMeetingDetailById?id=5140164653844480%3F">
YOCSEF (WuHan 2020) </a> <br>

&bull;&nbsp;The reviewers for peer-review journals, such as:
Bioinformatics, GPB,
TCBB, JTB, AIIM, SREP, BMC Bioinformatics, BMC Genomics, BMC Cancer,
BMC Systems Biology,
CBM, MBS, Methods, IET SYB, IJBRA, IJMI, IJDMB, Heliyon, IEEE Access,
Plos CB, Computational and Structural Biotechnology Journal, Journal of
Cellular and Molecular Medicine, Computational Biology and Chemistry,
OMICS: A Journal of Integrative Biology, etc <br>

&bull;&nbsp;The program committee member of <a href="http://www.aporc.org/ISB/2013/">ISB 2013</a>
, <a href="http://www.aporc.org/ISB/2014/">ISB 2014</a>
, <a href="http://www.aporc.org/ISB/2015/">ISB 2015</a>
, <a href="http://www.aporc.org/ISB/2016/">ISB 2016</a>
, <a href="http://www.aporc.org/ISB/2017/">ISB 2017</a>
, <a href="https://ritchielab.psu.edu/index.php/tbc2017/">TBC
2017</a>
, <a href="http://datamining-web.it.uts.edu.au/bioinformatics/committees.html">InCob
2017</a> ,<a href="http://www.aporc.org/ISB/2018/Committees.php">ISB
2018</a> , <a href="http://ic-ic.tongji.edu.cn/2018/index.htm">ICIC
2018</a> , <a href="http://orienta.ugr.es/bibm2018/">BIBM
2018</a> , <a href="http://ic-ic.tongji.edu.cn/2019/index.htm">ICIC
2019</a> , <a href="http://ieeebibm.org/BIBM2019/">BIBM
2019</a> </span></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE1"><span class="STYLE9">Projcts and awards:</span><br>

      </span><span class="STYLE4">&bull;&nbsp;NSFC
(31200987, 11871456). (PI) <br>

&bull;&nbsp;The Knowledge Innovation Program of SIBS CAS
(2013KIP218). (PI) <br>

&bull;&nbsp;The Natural Science Foundation of Shanghai
(17ZR1446100). (PI) <br>

&bull;&nbsp;The Opening Foundation of Shanghai Municipal
Science
and Technology Major Project (2017SHZDZX01). (PI) <br>

&bull;&nbsp;The Strategic Priority Research Program of CAS
(XDB13040700). <br>

&bull;&nbsp;The National Key R&amp;D Program - Special
Project on Precision Medicine (2016YFC0903400)). <br>

&bull;&nbsp;The National key research and development program
of China (2017YFA0505500). <br>

&bull;&nbsp;Project from Shanghai cooperative center for
transformation &amp; innovation. (Co-PI) <br>

&bull;&nbsp;Travel grant from BIBM2010 <br>

&bull;&nbsp;Travel grants from IDDF2018, IDDF2019 <br>

&bull;&nbsp;Best paper awards from ISB2013 <br>

&bull;&nbsp;Best paper awards from ISB2014 <br>

&bull;&nbsp;Best paper awards from BIBM2016 <br>

&bull;&nbsp;Best paper awards from InCob2017 <br>

&bull;&nbsp;Best poster awards from <a href="http://www.sysbio2019.org/index.php">1st national
systems biology conference (china,shanghai,2019)</a> <br>

      </span></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE1"><span class="STYLE9"><span class="STYLE8">Teaching
area and courses</span>:</span><br>

      </span><span class="STYLE4">Bioinformatics,
2013~2020 in SIBS</span>.</p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE11">Publication:</span></p>

      <p class="STYLE10"><em>Journal paper:</em></p>

      <p class="STYLE10">-------------------------------------------------------2021-----------------------------------------------------<br>

      <span class="STYLE3">[1]&nbsp; </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Yu XT, Chen Z. Applying artificial intelligence in the microbiome for
gastrointestinal diseases: A review. J Gastroenterol Hepatol. 2021
Apr;36(4):832-840. doi: 10.1111/jgh.15503.
(Corresponding author) </span></p>

      <p class="STYLE10"><span class="STYLE3">[2]&nbsp;
Guo WF, Zhang SW, Feng YH, Liang J, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L. Network controllability-based algorithm to target
personalized driver genes for discovering combinatorial drugs of
individual patients. Nucleic Acids Res. 2021 Apr 19;49(7):e37. doi:
10.1093/nar/gkaa1272.
(Co-corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[3]&nbsp;
Guo WF, Yu XT, Shi QQ, Liang J, Zhang SW, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
. Performance assessment of sample-specific network control methods for
bulk and single-cell biological data analysis. PLoS Comput Biol. 2021
May 6;17(5):e1008962. doi: 10.1371/journal.pcbi.1008962. Online ahead
of print.(Co-corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[4]&nbsp;
Zhang YH, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L, Huang T, Cai YD. Determining protein-protein functional
associations by functional rules based on gene ontology and KEGG
pathway. Biochim Biophys Acta Proteins Proteom. 2021
Jun;1869(6):140621. doi: 10.1016/j.bbapap.2021.140621. Epub 2021 Feb 6.
      </span> </p>

      <p class="STYLE10"><span class="STYLE3">[5]&nbsp;
Yuan F, Li Z, Chen L, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Ding S, Huang T, Cai YD. Identifying the Signatures and
Rules of Circulating Extracellular MicroRNA for Distinguishing Cancer
Subtypes. Front Genet. 2021 Mar 9;12:651610. doi:
10.3389/fgene.2021.651610. eCollection 2021. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[6]&nbsp;
Zhang YH, Li H, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L, Li Z, Huang T, Cai YD. Identifying Transcriptomic Signatures
and Rules for SARS-CoV-2 Infection. Front Cell Dev Biol. 2021 Jan
11;8:627302. doi: 10.3389/fcell.2020.627302. eCollection 2020. </span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[7]&nbsp;
Zhang YH, Li Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L, Li H, Gamarra M, Mansour RF, Escorcia-Gutierrez J, Huang T,
Cai YD. Investigating gene methylation signatures for fetal intolerance
prediction. PLoS One. 2021 Apr 22;16(4):e0250032. doi:
10.1371/journal.pone.0250032. eCollection 2021. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[8]&nbsp;
Pan XY, Li H, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Li Z, Chen L, Huang T, Cai YD. Identification of Protein Subcellular
Localization With Network and Functional Embeddings. Front Genet. 2021
Jan 20;11:626500. doi: 10.3389/fgene.2020.626500. eCollection 2020. </span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[9]&nbsp;
Chen L, Li Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Li H, Huang T, Cai YD. Predicting gene phenotype by
multi-label multi-class model based on essential functional features.
Mol Genet Genomics. 2021 Apr 29. doi: 10.1007/s00438-021-01789-8.
Online ahead of print. </span></p>

      <p class="STYLE10"><span class="STYLE3">[10]&nbsp;
Zhang YH, Li Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L, Li H, Huang T, Cai YD. Detecting the Multiomics Signatures of
Factor-Specific Inflammatory Effects on Airway Smooth Muscles. Front
Genet. 2021 Jan 13;11:599970. doi: 10.3389/fgene.2020.599970.
eCollection 2020. </span></p>

      <p class="STYLE10">-------------------------------------------------------2020-----------------------------------------------------<br>

      <span class="STYLE3">[1]&nbsp; Hu J, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Xia Q, Huang L, Zhang Y, Zhang C, Zeng Y, Liu H, Zhang S, Huang G,
Wan W, Ding Y, Hu F, Yang C, Chen L, Wang W. Identification of Key
Genes for the Ultrahigh Yield of Rice Using Dynamic Cross-tissue
Network Analysis. Genomics Proteomics Bioinformatics. 2020 Jul
28:S1672-0229(20)30091-7. doi: 10.1016/j.gpb.2019.11.007.
(Co-first author) </span></p>

      <p class="STYLE10"><span class="STYLE3">[2]&nbsp;
Yu X, Wang Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
. Essential gene expression pattern of head and neck squamous cell
carcinoma revealed by tumor-specific expression rule based on
single-cell RNA sequencing. Biochim Biophys Acta Mol Basis Dis. 2020
Mar 28;1866(7):165791. doi: 10.1016/j.bbadis.2020.165791.
(Co-corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[3]&nbsp;
Tang H, Tang Y, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L. Gene expression analysis reveals the tipping points during
infant brain development for human and chimpanzee. BMC Genomics. 2020
Mar 5;21(Suppl 1):74. doi: 10.1186/s12864-020-6465-8. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[4]&nbsp;
      </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Huang T, Lu C. Editorial: Machine Learning Advanced Dynamic Omics Data
Analysis for Precision Medicine. Front Genet. 2020 Feb 4;10:1343. doi:
10.3389/fgene.2019.01343. eCollection 2019. (Corresponding author) </span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[5]&nbsp;
Zhang S, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Hu B, Zhang YH, Feng K, Chen L, Niu Z, Li J, Huang T, Cai YD.
Discriminating Origin Tissues of Tumor Cell Lines by Methylation
Signatures and Dys-Methylated Rules. Front Bioeng Biotechnol. 2020 May
26;8:507. doi: 10.3389/fbioe.2020.00507. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[6]&nbsp;
Zhang S, Zhao J, Lv X, Fan J, Lu Y, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Wu H, Chen L, Zhao Y.
Analysis on gene modular network reveals morphogen-directed development
robustness in Drosophila.Cell Discov. 2020 Jun 30;6:43. doi:
10.1038/s41421-020-0173-z.
eCollection 2020. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[7]&nbsp;
Li Y, Omori A, Flores RL, Satterfield S, Nguyen C, Ota T, Tsurugaya T,
Ikuta T, Ikeo K, Kikuchi M, Leong JCK, Reich A, Hao M, Wan W, Dong Y,
Ren Y, Zhang S, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Uesaka M, Uchida Y, Li X, Shibata TF, Bino T,
Ogawa K, Shigenobu S, Kondo M, Wang F, Chen L, Wessel G, Saiga H,
Cameron RA, Livingston B, Bradham C, Wang W, Irie N.Genomic insights of
body plan transitions from bilateral to pentameral symmetry in
Echinoderms. Commun Biol. 2020 Jul 10;3(1):371. doi:
10.1038/s42003-020-1091-1. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[8]&nbsp;
Li ML, Tang H, Shao Y, Wang MS, Xu HB, Wang S, Irwin DM, Adeola AC, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Chen L, Li Y, Wu DD. Evolution and transition of expression trajectory
during human brain development. BMC Evol Biol. 2020 Jun 23;20(1):72.
doi: 10.1186/s12862-020-01633-4. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[9]&nbsp;
Yuan F, Pan X, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Zhang YH, Chen L, Gan Z, Huang T, Cai YD. Identifying Cell-Type
Specific Genes and Expression Rules Based on Single-Cell Transcriptomic
Atlas Data. Front Bioeng Biotechnol. 2020 Apr 29;8:350. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[10]&nbsp;
Li M, Pan X, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Feng K, Chen L, Huang T, Cai YD. Alternative
Polyadenylation Modification Patterns Reveal Essential
Posttranscription Regulatory Mechanisms of Tumorigenesis in Multiple
Tumor Types. Biomed Res Int. 2020 Jun 15;2020:6384120. doi:
10.1155/2020/6384120. eCollection 2020.</span> </p>

      <p class="STYLE10"><span class="STYLE3">[11]&nbsp;
Zhang Y, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Chen L, Ding S, Huang T, Cai YD. Identification of COVID-19
Infection-Related Human Genes Based on a Random Walk Model in a
Virus-Human Protein Interaction Network. Biomed Res Int. 2020 Jul
8;2020:4256301. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[12]&nbsp;
Pan X, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Chen L, Feng K, Huang T, Cai YD. Investigation
and Prediction of Human Interactome Based on Quantitative Features.
Front Bioeng Biotechnol. 2020 Jul 17;8:730. doi:
10.3389/fbioe.2020.00730. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[13]&nbsp;
Zhang YH, Pan XY, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L, Huang T, Cai YD. Identifying the RNA signatures of coronary
artery disease from combined lncRNA and mRNA expression profiles.
Genomics. 2020
Sep 9;S0888-7543(20)30508-5. doi: 10.1016/j.ygeno.2020.09.016. </span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[14]&nbsp;
Pan XY, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Chen L, Feng KY, Huang T, Cai YD. Investigation and
Prediction of Human Interactome Based on Quantitative Features. Front
Bioeng Biotechnol. 2020 Jul 17;8:730. doi: 10.3389/fbioe.2020.00730.</span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[15]&nbsp;
Chen L, Li Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Liu D, Li H, Huang T, Cai YD. Identifying Robust Microbiota
Signatures and Interpretable Rules to Distinguish Cancer Subtypes.
Front Mol Biosci. 2020 Nov 4;7:604794. doi: 10.3389/fmolb.2020.604794.</span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[16]&nbsp;
Zhang YH, Li Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Pan X, Chen L, Liu D, Li H, Huang T, Cai YD. Distinguishing
Glioblastoma Subtypes by Methylation Signatures. Front Genet. 2020 Nov
24;11:604336.</span> </p>

      <p class="STYLE10">-------------------------------------------------------2019-----------------------------------------------------<br>

      <span class="STYLE3">[1]&nbsp; Guo WF, Zhang SW,
      </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Akutsu T, Chen L. Network control principles for identifying
personalized driver genes in cancer. Brief Bioinform. 2019 Nov 10. pii:
bbz089. doi: 10.1093/bib/bbz089. (Co-corresponding author) </span></p>

      <p class="STYLE10"><span class="STYLE3">[2]&nbsp;
Guo W, Zhang SW, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Li Y, Gao J, Chen L. A novel network control model for identifying
personalized driver genes in cancer. PLoS Comput Biol. 2019 Nov
25;15(11):e1007520. doi: 10.1371/journal.pcbi.1007520. [Epub ahead of
print] (Co-corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[3]&nbsp;
      </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Dai H. Single-Cell RNA Sequencing-Based Computational Analysis to
Describe Disease Heterogeneity. Front Genet. 2019 Jul 12;10:629. doi:
10.3389/fgene.2019.00629. (Corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[4]&nbsp;
Lu Y, Fang Z, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Li M, Chen Q, Zhang H, Zhou Q, Hu Y, Chen L, Su S. Chronic hepatitis
B: dynamic change in Traditional Chinese Medicine syndrome by dynamic
network biomarkers. Chin Med. 2019 Nov 21;14:52. doi:
10.1186/s13020-019-0275-4. eCollection 2019. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[5]&nbsp;
Dai H, Li L, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Chen L. Cell-specific network constructed by single-cell RNA sequencing
data. Nucleic Acids Res. 2019 Mar 13. pii: gkz172. doi:
10.1093/nar/gkz172. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[6]&nbsp;
Tang H, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Chen L. High-Order Correlation Integration for Single-Cell or Bulk
RNA-seq Data
Analysis.
Front Genet. 2019 Apr 26;10:371. doi: 10.3389/fgene.2019.00371.
(Co-corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[7]&nbsp;
Gao W, Sun YB,
Zhou WW, Xiong ZJ, Chen L, Li H, Fu TT, Xu K, Xu W, Ma L, Chen YJ,
Xiang XY, Zhou L, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Zhang S, Jin JQ, Chen HM, Zhang G, Hillis DM, Ji X, Zhang YP, Che J.
Genomic and transcriptomic investigations of the evolutionary
transition from oviparity to viviparity. Proc Natl Acad Sci U S A. 2019
Feb 26;116(9):3646-3655. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[8]&nbsp;
Lu Y, Fang Z, Li M, Qian C, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Lu L, Chen Q, Zhang H, Zhou Q, Sun Y, Xue X, Hu Y, Chen L, Su S.
Dynamic edge-based biomarker noninvasively predicts hepatocellular
carcinoma with hepatitis B virus infection for individual patients
based on blood testing.J Mol Cell Biol. 2019 Mar 29. pii: mjz025. doi:
10.1093/jmcb/mjz025. [Epub ahead of print] </span> </p>

      <p class="STYLE10"><span class="STYLE3">[9]&nbsp;
Gao D, Zhang L, Song D, Lv J, Wang L, Zhou S, Li Y, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zeng Y,
Zhang J, Wang X. Values of integration between lipidomics and clinical
phenomes in patients with acute lung infection, pulmonary embolism, or
acute exacerbation of chronic pulmonary diseases: a preliminary study.
J Transl Med. 2019 May 20;17(1):162. doi: 10.1186/s12967-019-1898-z. </span>
      </p>

      <p class="STYLE10"><span class="STYLE3">[10]&nbsp;
Shi QQ, Hu B, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">and
Zhang CC. Multi-view Subspace Clustering Analysis for Aggregating
Multiple Heterogeneous Omics Data. Front. Genet., 20 August 2019 |
https://doi.org/10.3389/fgene.2019.00744 </span> </p>

      <p class="STYLE10"><span class="STYLE3">[11]&nbsp;
      </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Xiangtian Yu, Chengming Zhang.&#8197;Pan-cancer study of progressive
protein signatures for digestive cancers.Gut, 2019, 68 (Suppl 1)
A15-A16; DOI: 10.1136/gutjnl-2019-IDDFAbstracts.29. (ePoster)
(Corresponding author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[12]&nbsp;
Chen L, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Pan X, Zhang YH, Huang T and Cai YD. Identifying
Methylation Pattern and Genes Associated with Breast Cancer Subtypes.
Int. J. Mol. Sci. 2019, 20(17), 4269;
https://doi.org/10.3390/ijms20174269.
(Co-first
author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[13]&nbsp;
Song D, Tang L, Wang L, Huang J, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Fang H, Wang X. Roles of TGF&beta;1 in the expression of
phosphoinositide 3-kinase isoform genes and sensitivity and response of
lung telocytes to PI3K inhibitors. Cell Biol Toxicol. 2019 Sep 15. doi:
10.1007/s10565-019-09487-3. </span> </p>

      <p class="STYLE10"><span class="STYLE3">[14]&nbsp;
Chen L, Pan XY, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang YH, Huang T, Cai YD. Identifying essential signature genes and
expression rules associated with distinctive development stages of
early embryonic cells. IEEE Access, 2019, 7:128570 - 128578.
DOI:10.1109/ACCESS.2019.2939556.
(Co-first
author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[15]&nbsp;
Pan XY, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Yuan F, Zhang YH, Chen L, Zhu LC, Wan SB, Huang T, Cai YD. Screening of
Methylation Signature and Gene Functions Associated With the Subtypes
of Isocitrate Dehydrogenase-Mutation Gliomas. Frontiers in
Bioengineering and Biotechnology, 2019, 7:339.
DOI=10.3389/fbioe.2019.00339. (Co-first
author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[16]&nbsp;
Chen L, Pan X, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Zhang Y-H, Zhang Y, Huang T and Cai Y-D. Immunosignature Screening
for Multiple Cancer Subtypes Based on Expression Rule. Front. Bioeng.
Biotechnol. 2019, 7:370. doi: 10.3389/fbioe.2019.00370 (Co-first
author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[17]&nbsp;
Choobdar S, Ahsen ME, Crawford J, ..., </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, ..., Sven Bergmann &amp; Daniel Marbach. Assessment of network
module identification across complex diseases. Nature Methods,
16:843&ndash;852 (2019). (Consortium author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[18]&nbsp;
Zhang S, Pan X, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Guo W, Gan Z, Zhang YH, Chen L, Zhang Y, Huang T, Cai YD. Copy Number
Variation Pattern for Discriminating MACROD2 States of Colorectal
Cancer Subtypes. Front Bioeng Biotechnol. 2019 Dec 19;7:407. doi:
10.3389/fbioe.2019.00407. eCollection 2019.
(Co-first
author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[19]&nbsp;
Sun S, Yu X, Sun F, Tang Y, Zhao J, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
. Dynamically characterizing individual clinical change by the steady
state of disease-associated pathway. BMC Bioinformatics. 2019 Dec
24;20(Suppl 25):697. doi: 10.1186/s12859-019-3271-x. (Co-corresponding
author) </span> </p>

      <p class="STYLE10"><span class="STYLE3">[20]&nbsp;
Song D, Tang L, Huang J, Wang L, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Wang X. Roles of transforming growth factor-&beta; and
phosphatidylinositol 3-kinase isoforms in integrin &beta;1-mediated
bio-behaviors of mouse lung telocytes. J Transl Med. 2019 Dec
30;17(1):431. doi: 10.1186/s12967-019-02181-2. </span> </p>

      <p class="STYLE10">-------------------------------------------------------2018-----------------------------------------------------<br>

      <span class="STYLE3">[1]&nbsp; Wang L, Yu X,
Zhang C, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
. Detecting Personalized Determinants During Drug Treatment from Omics
Big Data. Curr Pharm Des. 2018;24(32):3727-3738. doi:
10.2174/1381612824666181106102111.
(Corresponding author) </span></p>

      <p class="STYLE10"><span class="STYLE3">[2]&nbsp;
      </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">, Lu Wang, Xiangtian Yu
. A revisit on personalized critical variance of gut microbiota during
the occurrence of Type I Diabetes. Gut, 2018, 67(Suppl 2):A7.
(ePoster) (Corresponding author) </span> </p>

      <p class="STYLE10"> <span class="STYLE3">[3]&nbsp;
Shaoyan Sun,
Xiangtian Yu,
Fengnan Sun, Ying Tang, Juan Zhao, and </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
. Characterizing and Discriminating Individual Steady State of
Disease-Associated Pathway. Intelligent Computing Theories and
Application, LNCS 10954, 2018, pp.533-538. (Co-corresponding author) </span>
      </p>

      <p class="STYLE10"> <span class="STYLE3">[4]&nbsp;
Hu
J, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">
, Xia Q, Qian Q, Yang C, Ding Y, Chen L, Wang W.
Unravelling miRNA regulation in yield of rice (Oryza sativa) based on
differential network model. Scientific Reportsvolume 8, Article number:
8498, 2018. (Co-first
author) </span> </p>

      <p class="STYLE10"> <span class="STYLE3">[5]&nbsp;
Guo
Weifeng,
Zhang SW, Liu LL, Liu F, Shi QQ, Zhang L, Tang Y, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Chen L. Discovering personalized driver mutation profiles of single
samples in cancer by network control strategy. Bioinformatics. 2018 Jan
10. doi: 10.1093/bioinformatics/bty006. [Epub ahead of print]
(Co-corresponding author)</span> <br>

      </p>

      <p class="STYLE10"> <span class="STYLE3">[6]&nbsp;Guo
Weifeng, Zhang SW, Shi QQ,
Zhang CM, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">,
Chen L. A novel algorithm for finding optimal driver nodes to target
control complex networks and its applications for drug targets
identification.&nbsp;&nbsp;&nbsp; BMC Genomics. 2018 Jan
19;19(Suppl
1):924. doi: 10.1186/s12864-017-4332-z. (Co-corresponding author) </span><br>

      </p>

      <p class="STYLE10"> <span class="STYLE3">[7]&nbsp;
Zhang JS, Guo JM, Zhang M, Yu XT, Yu XQ, Guo WF, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">, Chen LN. Efficient Mining Multi-mers in a
Variety of Biological Sequences.
doi:10.1109/TCBB.2018.2828313. (Co-corresponding author) </span><br>

      </p>

      <p class="STYLE10"> <span class="STYLE3">[8]&nbsp;
Yu XT, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">. Integrative Analysis of Omics Big Data.
Methods Mol Biol. 2018;1754:109-135. doi: 10.1007/978-1-4939-7717-8_7.
(Chapter) </span><br>

      </p>

      <p class="STYLE10"> <span class="STYLE3">[9]&nbsp;
Yu XT, Wang L, </span><span class="STYLE3" style="font-weight: bold;">Zeng T</span><span class="STYLE3">. Revisit of Machine Learning Supported
Biological and Biomedical Studies.
Methods Mol Biol. 2018;1754:183-204. doi: 10.1007/978-1-4939-7717-8_11.
(Chapter) </span><br>

      </p>

      <p class="STYLE10"> <br>

-------------------------------------------------------2017-----------------------------------------------------<br>

      <span class="STYLE12">[1] Yu X, Zhang J, Sun S, Zhou
X, <strong>Zeng T</strong>, Chen L. Individual-specific
edge-network analysis for disease prediction. Nucleic Acids Res. 2017
Nov 16;45(20):e170. doi: 10.1093/nar/gkx787. PubMed
PMID: 28981699. (Co-corresponding author) </span></p>

      <p class="STYLE3">[2] Shi Q, Zhang C, Peng M, Yu X, <strong>Zeng
T</strong>, Liu J, Chen L. Pattern fusion analysis by adaptive
alignment of multiple heterogeneous omics data. Bioinformatics. 2017
Sep 1;33(17):2706-2714. doi: 10.1093/bioinformatics/btx176. PubMed
PMID: 28520848. (Co-corresponding author)</p>

      <p class="STYLE3">[3] Zhang C, Liu J, Shi Q, <strong>Zeng
T</strong>
, Chen L. Comparative network stratification analysis for identifying
functional interpretable network biomarkers. BMC Bioinformatics. 2017
Mar 14;18(Suppl 3):48. doi: 10.1186/s12859-017-1462-x. PubMed
PMID: 28361683. (Co-corresponding author) </p>

      <p class="STYLE3">[4] Li M, Li C, Liu WX, Liu C, Cui
J, Li Q, Ni H, Yang Y, Wu C, Chen C, Zhen X, <strong>Zeng
T</strong>
, Zhao M, Chen L, Wu J, Zeng R, Chen L. Dysfunction of PLA2G6 and
CYP2C44-associated network signals imminent carcinogenesis from chronic
inflammation to hepatocellular carcinoma. J Mol Cell Biol. 2017 Jun 26.
doi: 10.1093/jmcb/mjx021. [Epub ahead of print] PubMed
PMID: 28655161. </p>

      <p class="STYLE3">[5] Shi Q, Zhang C, Guo W, <strong>Zeng
T</strong>
, Lu L, Jiang Z, Wang Z, Liu J, Chen L. Local network component
analysis for quantifying transcription factor activities. Methods. 2017
Jul 15;124:25-35. doi: 10.1016/j.ymeth.2017.06.018. Epub 2017 Jul 12.
PubMed
PMID: 28710010. )</p>

      <p class="STYLE3">[7] Zhang C, Liu J, Shi QQ, <strong>Zeng
T</strong>
, Chen L. Differential function analysis: identifying structure
and activation variations in dysregulated pathways. SCIENCE CHINA
Information Sciences, Vol. 60 012108:1&ndash;012108:16, 2017.
(Co-corresponding author) </p>

      <p class="STYLE10">-------------------------------------------------------2016-----------------------------------------------------<br>

      <span class="STYLE12">[1] Yang Y, Lin X, Lu X, Luo
G, <strong>Zeng T</strong>, Tang J, Jiang F, Li L, Cui X,
Huang W, Hou G, Chen X, Ouyang Q, Tang S, Sun H, Chen L, Gonzalez FJ,
Wu M, Cong W, Chen L, Wang H. Interferon-microRNA signalling drives
liver precancerous lesion formation and hepatocarcinogenesis. Gut. 2016
Feb 9. pii: gutjnl-2015-310318. doi: 10.1136/gutjnl-2015-310318. [Epub
ahead of print] PubMed PMID: 26860770.</span></p>

      <p class="STYLE10">-------------------------------------------------------2015-----------------------------------------------------<br>

      <span class="STYLE12">[1] <strong>Zeng T</strong>,
Zhang W, Yu X, Liu X, Li M, Chen L. Big-data-based edge biomarkers:
study on dynamical drug sensitivity and resistance in individuals.
Brief Bioinform. 2015 Sep 26. pii: bbv078. [Epub ahead of print] PubMed
PMID: 26411472.</span></p>

      <p class="STYLE3">[2] Yu X, <strong>Zeng T</strong>,
Li G. Integrative enrichment analysis: a new computational method to
detect dysregulated pathways in heterogeneous samples. BMC Genomics.
2015 Nov 10;16:918. doi: 10.1186/s12864-015-2188-7. PubMed PMID:
26556243; PubMed Central PMCID: PMC4641376. (Co-corresponding author)</p>

      <p class="STYLE3">[3] Kang H, Cho KH, Zhang XD, <strong>Zeng
T</strong>, Chen L. Inferring Sequential Order of Somatic
Mutations during Tumorgenesis based on Markov Chain Model. IEEE/ACM
Trans Comput Biol Bioinform. 2015 Sep-Oct;12(5):1094-103. doi:
10.1109/TCBB.2015.2424408. PubMed PMID: 26451822. (Co-corresponding
author)</p>

      <p class="STYLE3">[4] Yu X, <strong>Zeng T</strong>,
Wang X, Li G, Chen L. Unravelling personalized dysfunctional gene
network of complex diseases based on differential network model. J
Transl Med. 2015 Jun 13;13:189. doi: 10.1186/s12967-015-0546-5. PubMed
PMID: 26070628; PubMed Central PMCID: PMC4467679.</p>

      <p class="STYLE3">[5] Shi Q, Liu X, <strong>Zeng
T</strong>, Wang W, Chen L. Detecting disease genes of non-small
lung cancer based on consistently differential interactions. Cancer
Metastasis Rev. 2015 Jun;34(2):195-208. doi: 10.1007/s10555-015-9561-5.
PubMed PMID: 26004969.</p>

      <p class="STYLE3">[6] Yang D, Zhou J,<strong>
Zeng T</strong>, Yang Z, Wang X, Hu J, Song Y, Chen L, Peer D,
Wang X, Bai C. Serum chemokine network correlates with chemotherapy in
non-small cell lung cancer. Cancer Lett. 2015 Aug 28;365(1):57-67. doi:
10.1016/j.canlet.2015.05.001. Epub 2015 May 11. PubMed PMID: 25976768.</p>

      <p class="STYLE3">[7] Chen S, Liu J, <strong>Zeng
T</strong>. Measuring the quality of linear patterns in
biclusters. Methods. 2015 Jul 15;83:18-27. doi:
10.1016/j.ymeth.2015.04.005. Epub 2015 Apr 15. PubMed PMID: 25890245.</p>

      <p class="STYLE3">[8] Zhang W, <strong>Zeng T</strong>,
Liu X, Chen L. Diagnosing phenotypes of single-sample individuals by
edge biomarkers. J Mol Cell Biol. 2015 Jun;7(3):231-41. doi:
10.1093/jmcb/mjv025. Epub 2015 Apr 26. PubMed PMID: 25917597. (Co-first
author)</p>

      <p class="STYLE10">-------------------------------------------------------2014-----------------------------------------------------<br>

      <span class="STYLE12">[1] <strong>Zeng T</strong>,
Zhang W, Yu X, Liu X, Li M, Liu R, Chen L. Edge biomarkers for
classification and prediction of phenotypes. Sci China Life Sci. 2014
Nov;57(11):1103-14. doi: 10.1007/s11427-014-4757-4. Epub 2014 Oct 17.
Review. PubMed PMID: 25326072.</span></p>

      <p class="STYLE3">[2] Wen Z, Zhang W,<strong>
Zeng T</strong>, Chen L. MCentridFS: a tool for identifying
module biomarkers for multi-phenotypes from high-throughput data. Mol
Biosyst. 2014 Nov;10(11):2870-5. doi: 10.1039/c4mb00325j. PubMed PMID:
25099602.</p>

      <p class="STYLE3">[3] <strong>Zeng T</strong>,
Wang DC, Wang X, Xu F, Chen L. Prediction of dynamical drug sensitivity
and resistance by module network rewiring-analysis based on
transcriptional profiling. Drug Resist Updat. 2014 Jul;17(3):64-76.
doi: 10.1016/j.drup.2014.08.002. Epub 2014 Aug 13. Review. PubMed PMID:
25156319.</p>

      <p class="STYLE3">[4] Zhang W, <strong>Zeng T</strong>,
Chen L. EdgeMarker: Identifying differentially correlated molecule
pairs as edge-biomarkers. J Theor Biol. 2014 Dec 7;362:35-43. doi:
10.1016/j.jtbi.2014.05.041. Epub 2014 Jun 12. PubMed PMID: 24931676.
(Co-corresponding author)</p>

      <p class="STYLE3">[5] <strong>Zeng T</strong>,
Zhang CC, Zhang W, Liu R, Liu J, Chen L. Deciphering early development
of complex diseases by progressive module network. Methods. 2014 Jun
1;67(3):334-43. doi: 10.1016/j.ymeth.2014.01.021. Epub 2014 Feb 21.
PubMed PMID: 24561825.</p>

      <p class="STYLE3">[6] Zhao M, Chang HT, Zhou Q, <strong>Zeng
T</strong>, Shih CS, Liu ZP, Chen L, Wei DQ. Predicting
protein-ligand interactions based on chemical preference features with
its application to new D-amino acid oxidase inhibitor discovery. Curr
Pharm Des. 2014;20(32):5202-11. PubMed PMID: 24410568.</p>

      <p class="STYLE3">[7] Li M, <strong>Zeng T</strong>,
Liu R, Chen L. Detecting tissue-specific early warning signals for
complex diseases based on dynamical network biomarkers: study of type 2
diabetes by cross-tissue analysis. Brief Bioinform. 2014
Mar;15(2):229-43. doi: 10.1093/bib/bbt027. Epub 2013 Apr 25. PubMed
PMID: 23620135. (Co-first author)</p>

      <p class="STYLE10">------------------------------------------------2013
and before-------------------------------------------------<br>

      <span class="STYLE12">[1]<strong> Zeng T</strong>,
Sun SY, Wang Y, Zhu H, Chen L. Network biomarkers reveal dysfunctional
gene regulations during disease progression. FEBS J. 2013
Nov;280(22):5682-95. doi: 10.1111/febs.12536. Epub 2013 Oct 22. Review.
PubMed PMID: 24107168.</span></p>

      <p class="STYLE3">[2] Tarca AL, Lauria M, Unger M,
Bilal E, Boue
S, Kumar Dey K, Hoeng J, Koeppl H, Martin F, Meyer P, Nandy P, Norel R,
Peitsch M, Rice JJ, Romero R, Stolovitzky G, Talikka M, Xiang Y,
Zechner C; IMPROVER DSC Collaborators. Strengths and limitations of
microarray-based phenotype prediction: lessons learned from the
IMPROVER Diagnostic Signature Challenge. Bioinformatics. 2013 Nov
15;29(22):2892-9. doi: 10.1093/bioinformatics/btt492. Epub 2013 Aug 20.
(Collaborators) </p>

      <p class="STYLE3">[3] Sun SY, Liu ZP, <strong>Zeng
T</strong>, Wang Y, Chen L. Spatio-temporal analysis of type 2
diabetes mellitus based on differential expression networks. Sci Rep.
2013;3:2268. doi: 10.1038/srep02268. PubMed PMID: 23881262; PubMed
Central PMCID: PMC3721080.</p>

      <p class="STYLE3">[4] <strong>Zeng T</strong>,
Chen L. Tracing dynamic biological processes during phase transition.
BMC Syst Biol. 2012;6 Suppl 1:S12. doi: 10.1186/1752-0509-6-S1-S12.
Epub 2012 Jul 16. Erratum in: BMC Syst Biol. 2012 Jul 16;6 Suppl 1:S23.
PubMed PMID: 23046764; PubMed Central PMCID: PMC3403121.</p>

      <p class="STYLE3">[5] Xiong Y, Liu J, Zhang W, <strong>Zeng
T</strong>. Prediction of heme binding residues from protein
sequences with integrative sequence profiles. Proteome Sci. 2012 Jun
21;10 Suppl 1:S20. doi: 10.1186/1477-5956-10-S1-S20. PubMed PMID:
22759579; PubMed Central PMCID: PMC3380730.</p>

      <p class="STYLE3">[6]<strong> Zeng T</strong>,
Li J, Liu J. Distinct interfacial biclique patterns between
ssDNA-binding proteins and those with dsDNAs. Proteins. 2011
Feb;79(2):598-610. doi: 10.1002/prot.22908. PubMed PMID: 21120860.</p>

      <p class="STYLE3">[7]<strong> Zeng T</strong>,
Li J. Maximization of negative correlations in time-course gene
expression data for enhancing understanding of molecular pathways.
Nucleic Acids Res. 2010 Jan;38(1):e1. doi: 10.1093/nar/gkp822. Epub
2009 Oct 23. PubMed PMID: 19854949; PubMed Central PMCID: PMC2800212.</p>

      <p class="STYLE3">[8] <strong>Zeng T</strong>,
Liu J. Mixture classification model based on clinical markers for
breast cancer prognosis. Artif Intell Med. 2010 Feb-Mar;48(2-3):129-37.
doi: 10.1016/j.artmed.2009.07.008. Epub 2009 Dec 14. PubMed PMID:
20005686.</p>

      <p class="STYLE3">[9] Gu C, <strong>Zeng T</strong>,
Li Y, Xu Z, Mo Z, Zheng C. Structure-function analysis of mutant
RNA-dependent RNA polymerase complexes with VPg. Biochemistry (Mosc).
2009 Oct;74(10):1132-41. PubMed PMID: 19916926. (Co-first author) </p>

      <p class="STYLE3">[10] <strong>Zeng T</strong>,
Zhongxi Mo. Bioinformatics analysis of mutant structure and functional
sites of foot-and-mouth disease virus RNA-dependent RNA polymerase.
Journal of Computational and Theoretical Nanoscience, v 4, n 7-8, p
1330-1336, November/December 2007. </p>

      <p class="STYLE3">[11] Zhongxi Mo, <strong>Zeng
T</strong> . Improved genetic algorithm for problem of genome
rearrangement. Wuhan University Journal of Natural Sciences, v 11, n 3,
p 498-502, May 2006.<br>

      </p>

      <p class="STYLE3">&nbsp;</p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><em>Conference paper:</em><br>

      <span class="STYLE12">[1] Lu wang, Xiangtian Yu, <strong>Tao
Zeng</strong>.
A revisit on personalized critical variance of gut microbiota during
the occurrence of Type I Diabetes. IDDF 2018. (ePoster) </span></p>

      <p class="STYLE3">[2] <strong>Tao Zeng</strong>.
Improving the
discrimination and interpretability for complex diseases by network
biomarker. 16th SCBA International Symposium, 2017, pp.298. (Abstract) </p>

      <p class="STYLE3"> [3] Weifeng Guo, Shaowu Zhang,
Qianqian Shi, Chengming Zhang, <strong>Tao Zeng</strong>
and Luonan Chen . A novel algorithm for finding optimal driver nodes to
target control complex networks and its applications for drug targets
identification. International Conference on Bioinformatics (InCoB)
2017. </p>

      <p class="STYLE3">[4] Chuanchao Zhang, Juan Liu,
Qianqian Shi, <strong>Tao Zeng</strong> and Luonan Chen.
Comparative network stratification analysis for identifying functional
interpretable network biomarkers. The Asia Pacific Bioinformatics
Conference (APBC) 2017. </p>

      <p class="STYLE3"> [5] Chuanchao Zhang, Juan Liu, <strong>Tao
Zeng</strong>, and Luonan Chen. Integration of multiple
heterogeneous omics data
.The IEEE International Conference on Bioinformatics and Biomedicine
(BIBM) 2016. </p>

      <p class="STYLE3">[6] Chuanchao Zhang, Juan Liu,
Qianqian Shi, <strong>Tao Zeng</strong>, Luonan Chen.
Identification of phenotypic networks based on whole transcriptome by
comparative network decomposition. 2015 IEEE International Conference
on Bioinformatics and Biomedicine (BIBM), pp: 189-194. </p>

      <p class="STYLE3">[7] <strong>Tao Zeng</strong>,
Ziming Wang, Luonan Chen: Detecting Phosphorylation Determined Active
Protein Interaction Network during Cancer Development by Robust Network
Component Analysis. DTMBIO '14 Proceedings of the ACM 8th International
Workshop on Data and Text Mining in Bioinformatics ,DTMBIO@CIKM 2014,
pp.49. (Abstract)</p>

      <p class="STYLE3">[8] <strong>Tao Zeng</strong>,
Ruozhao Wang, Luonan Chen. Module network rewiring in response to
therapy. IEEE 6th International Conference on Systems Biology, ISB
2012, p 197-202.</p>

      <p class="STYLE3">[9] <strong>Tao Zeng</strong>,
Luonan Chen. Identifying temporal trace of biological process during
phase transition. Systems Biology (ISB), 2011 IEEE International
Conference on, pp:368-373.</p>

      <p class="STYLE3">[10] Guanying Piao, Bangguo Qian,
Shigeru Saito, ZP Liu, <strong>Tao Zeng</strong>, Y. Wang,
Jiarui Wu, Huarong Zhou, L. Chen, K. Horimoto. Phenotype-difference
oriented identification of molecular functions for diabetes progression
in Goto-Kakizaki rat. Systems Biology (ISB), 2011 IEEE International
Conference on, pp:111-116.</p>

      <p class="STYLE3">[11] Yi Xiong, Wen Zhang, <strong>Tao
Zeng</strong>, Juan Liu. Prediction of heme binding sites in heme
proteins using an integrative sequence profile coupling evolutionary
information with physicochemical properties. Proceedings - 2011 IEEE
International Conference on Bioinformatics and Biomedicine, BIBM 2011,
p 143-146, 2011.</p>

      <p class="STYLE3">[12] Jinyan Li, Qian Liu, <strong>Tao
Zeng</strong>. Negative correlations in collaboration: concepts
and algorithms. Knowledge Discovery and Data Mining, KDD2010, p
463-472, 2010.</p>

      <p class="STYLE3">[13] <strong>Tao Zeng</strong>,
Xuan Guo, Juan Liu. Discovering negative correlated gene sets from
integrative gene expression data for cancer prognosis. Proceedings -
2010 IEEE International Conference on Bioinformatics and Biomedicine,
BIBM 2010, p 489-492, 2010, </p>

      <p class="STYLE3">[14] <strong>Tao Zeng</strong>,
Fei Luo, Juan Liu. Biological pathway conducting microarray-based
cancer classification. Proceedings of the 2009 2nd International
Conference on Biomedical Engineering and Informatics, BMEI 2009, 2009.</p>

      <p class="STYLE3">[15] <strong>Tao Zeng</strong>,
Juan Liu. A novel mixture classifier and its application in breast
cancer prognosis. 2nd International Conference on Bioinformatics and
Biomedical Engineering, iCBBE 2008, p 564-568, 2008.</p>

      <p class="STYLE3">[16] <strong>Tao Zeng</strong>,
Juan Liu. Analysis on time-lagged gene clusters in time series gene
expression data. Proceedings - 2007 International Conference on
Computational Intelligence and Security, CIS 2007, p 182-185, 2007.</p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><em>Chinese book &amp;
chapter:</em><br>

      <span class="STYLE12">[1] &#38472;&#27931;&#21335;, <strong>&#26366;&#28059;</strong>.
2.13 &#22823;&#25968;&#25454;&#26102;&#20195;&#30340;&#29983;&#29289;&#20449;&#24687;&#23398;&#19982;&#31995;&#32479;&#29983;&#29289;&#23398;&#26032;&#36827;&#23637;. &#20013;&#22269;&#31185;&#23398;&#38498;&#12298;2016&#31185;&#23398;&#21457;&#23637;&#25253;&#21578;&#12299;, ISBN&#65306;978-7-03-048895-4. </span></p>

      <span class="STYLE12">[2] &#21016;&#27835;&#24179;, &#36213;&#20852;&#26126;, <strong>&#26366;&#28059;</strong>,
&#38472;&#27931;&#21335;. &#31532;&#20843;&#31456;&#29983;&#29289;&#22823;&#20998;&#23376;&#20449;&#21495;&#20256;&#23548;&#21644;&#32593;&#32476;. &#20013;&#22269;&#23398;&#31185;&#21457;&#23637;&#25112;&#30053;-&#29702;&#35770;&#19982;&#35745;&#31639;&#21270;&#23398;, ISBN&#65306;9787030489197. </span>
      <p></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE8">Softwares:</p>

      <p class="STYLE4">&hearts;&nbsp;MCentridFS (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/MCentridFS.rar">Multi-class
Centroid Feature Selection</a>): supply a tool for the optimal
module analysis on the biological active network </p>

      <p class="STYLE4">&hearts;&nbsp;PMNminer (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/PMNminer.rar">Progressive
module network analysis</a>): supply a computational framework
for the cross-tissue analysis on the phenotype-dynamics of biological
process (e.g. disease occurance and progression) </p>

      <p class="STYLE4">&hearts;&nbsp;IEApackage (<a href="https://github.com/bluesky2009/integrative-enrichment-analysis">Integrative
enrichment analysis</a>): supply a software package for the
functional enrichment analysis on the heterogeneous samples </p>

      <p class="STYLE4">&hearts;&nbsp;PFA (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/PFApackage_0.1.rar">Pattern
fusion analysis</a>): supply an integrative tool to identify
sample-patterns from different omics profiles by optimally adjusting
the effects of each data type to the hidden patterns. </p>

      <p class="STYLE4">&hearts;&nbsp;CNS (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/CNSpackage_0.1.rar">Comparative
Network Stratification</a>): supply a computational methods to
directly identify functional network biomarkers with both
discriminative power on disease states and readable interpretation on
biological functions. </p>

      <p class="STYLE4">&hearts;&nbsp;DFApackage (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/DFApackage.zip">Differential
function analysis</a>): supply a general framework to identify
the changes of element-structure and expression-activation of
biological functions, based on comparative non-negative matrix
factorization (cNMF). </p>

      <p class="STYLE4">&hearts;&nbsp;SCS (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/supplemntary_MATERIAL.zip">Single-sample
Controller Strategy</a>): supply a new single-sample controller
strategy (SCS) to identify personalized driver mutation profiles from
network controllability perspective. </p>

      <p class="STYLE4">&hearts;&nbsp;eTMpy (<a href="https://github.com/ztpub/eTMpy">Endogenous target
mimic (eTMs) screen package</a>): supply a python script for
screening the endogenous target mimic in plant. </p>

      <p class="STYLE3">&nbsp;</p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE1"><span class="STYLE9"><span class="STYLE8">Photos</span>:</span><br>

      </span></p>

      <span class="STYLE4"><a href="http://taulab.cn/images/2019-1.jpg">Mid-term report
meeting
(2019)</a></span> <span class="STYLE4"><a href="http://taulab.cn/images/2019-2.jpg">CAS Life Science
Forum (2019)</a></span> <span class="STYLE4"><a href="http://taulab.cn/images/CBC2019.jpg">The Fourth CCF
Bioinformatics Conference (2019)</a></span> <span class="STYLE4"><a href="http://taulab.cn/images/ChinaSBC-shanghai2019.JPG">The
first Chinese Systems Biology Conference (2019)</a></span>
      <p></p>

      <p></p>

      </td>

    </tr>

  </tbody>
</table>

<br>

<!-- bottom menu table -->
<!-- bottom menu table --><font face="Verdana" size="1">&copy;2018 Homepage of Tao Zeng <b>
&tau;&trade;</b></font><br>

</div>

<span style="display: none;"><a href="http://www.mianfeimoban.com">mianfeimoban</a></span>
<script language="JavaScript">
<!--
var caution = false
function setCookie(name, value, expires, path, domain, secure){
var curCookie = name + "=" + escape(value) +
((expires) ? "; expires=" + expires.toGMTString() : "") +
((path) ? "; path=" + path : "") +
((domain) ? "; domain=" + domain : "") +
((secure) ? "; secure" : "")
if (!caution || (name + "=" + escape(value)).length<= 4000)
document.cookie = curCookie
else
if (confirm("Cookie exceeds 4KB and will be cut!"))
document.cookie = curCookie
}
function getCookie(name) {
var prefix = name + "="
var cookieStartIndex = document.cookie.indexOf(prefix)
if (cookieStartIndex == -1)
return null
var cookieEndIndex = document.cookie.indexOf(";", cookieStartIndex+ prefix.length)
if (cookieEndIndex == -1)
cookieEndIndex = document.cookie.length
return unescape(document.cookie.substring(cookieStartIndex +prefix.length, cookieEndIndex))
}
function deleteCookie(name, path, domain) {
if (getCookie(name)) {
document.cookie = name + "=" +
((path) ? "; path=" + path : "") +
((domain) ? "; domain=" + domain : "") +
"; expires=Thu, 01-Jan-70 00:00:01 GMT"
}
}
function fixDate(date) {
var base = new Date(0)
var skew = base.getTime()
if (skew > 0)
date.setTime(date.getTime() - skew)
}
var now = new Date()
fixDate(now)
now.setTime(now.getTime() + 365 * 24 * 60 * 60 * 1000)
var visits = getCookie("counter")
if (!visits)
visits = 1
else
visits = parseInt(visits) + 1
setCookie("counter", visits, now)
document.write("There are " + visits + " visitions&#65281;")
// -->
</script>
&gt;&gt;&gt;&gt;&gt;&gt;&gt;
ab8ebc2f8124e84f646db7ebc574880c35d9627a
</body>
</html>
