<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
  <title>HomePage of Dr. Tao Zeng (SIBS)</title>


  <style type="text/css">
<!--
.STYLE1 {
font-size: 18px;
font-weight: bold;
}
.STYLE3 {
font-size: 12px;
font-family: "Times New Roman", Times, serif;
}
.STYLE4 {
font-size: 14px;
font-family: "Times New Roman", Times, serif;
}
.STYLE8 {
font-size: 24px;
font-weight: bold;
font-family: "Times New Roman", Times, serif;
}
.STYLE9 {font-size: 24px}
.STYLE10 {font-family: "Times New Roman", Times, serif}
.STYLE11 {font-weight: bold; font-size: 24px;}
.STYLE12 {font-size: 12px}
-->
  </style>
</head>


<body topmargin="0" leftmargin="0" style="color: rgb(255, 255, 255); background-color: rgb(128, 128, 192);" alink="#ccccff" link="#ccccff" marginheight="0" marginwidth="0" vlink="#ccccff">

<div align="center">
<h1><font face="Verdana" size="7">Dr. Tao Zeng</font></h1>

<!-- top menu table -->
<!-- top menu table --><br>

<table cellpadding="5" cellspacing="5">

  <tbody>

    <tr>

      <td valign="top" width="25%"><!-- left box table -->
      <table bgcolor="#ffffff" cellpadding="0" cellspacing="1">

        <tbody>

          <tr>

            <td>
            <table bgcolor="#a0a0e0" width="100%">

              <tbody>

                <tr>

                  <td> <font face="Verdana" size="1">
                  <b>News and Photos</b></font> </td>

                </tr>

              </tbody>
            </table>

            </td>

          </tr>

          <tr>

            <td>
            <table bgcolor="#9090d0" width="100%">

              <tbody>

                <tr>

                  <td> <font face="Verdana" size="1">Dr.
Tao Zeng in FuDan University ...2016.5</font><br>

                  <div style="text-align: justify;"><font face="Verdana" size="1">taken an talk about "network
biomarker" in ShangHai Bioinformatics seminar</font></div>

                  <font face="Verdana" size="1"><br>

Dr.
Tao Zeng in BeiJing ................2016.4</font><br>

                  <div style="text-align: justify;"><font face="Verdana" size="1">served
as the academic secretary of The 557th Session of XIANGSHAN-SCIENCE
Conferences: Big Data, Precision Medicine and Bioinformatics</font><br>

                  </div>

                  <font face="Verdana" size="1"><br>

Dr.
Tao Zeng in ShangHai .............2016.3<br>

                  <br>

                  <img style="width: 243px; height: 333px;" src="zt.jpeg"> <br>

                  <br>

                  </font> </td>

                </tr>

              </tbody>
            </table>

            </td>

          </tr>

        </tbody>
      </table>

<!-- left box table --> </td>

      <td valign="top">
      <p class="STYLE10"><span class="STYLE8">Associate
Professor</span><br>

      <span class="STYLE3"> &nbsp;&nbsp;Key
Laboratory of Systems Biology <br>

&nbsp;&nbsp;Shanghai Institutes for Biological Sciences<br>

&nbsp;&nbsp;Chinese Academy of Sciences</span></p>

      <p class="STYLE3"> &nbsp;&nbsp;Present
address (office): Cell building 333, 320 YueYang Road, Shanghai 200031,
China<br>

&nbsp;&nbsp;Phone (office): +86-21-5492-0102<br>

&nbsp;&nbsp;E-mail: zengtao(at)sibs.ac.cn<br>

&nbsp;&nbsp;Sites online: <a href="https://www.researchgate.net/profile/Tao_Zeng4">Researchgate</a>,
      <a href="http://www.researcherid.com/rid/A-1711-2012">Researcher
ID</a>, <a href="http://www.scholarmate.com/profile/taoceng36">ScholarMate</a>
, <a href="https://www.linkedin.com/in/tazeng">LinkedIn</a></p>

      <p></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE8">Education:</span><br>

      <span class="STYLE4">&bull;&nbsp;2006~2010 <a href="http://www.whu.edu.cn/">Wuhan University</a>
, <a href="http://cs.whu.edu.cn/">School of computer</a>,
Received the doctor's degree.<br>

&bull;&nbsp;2003~2006 <a href="http://www.whu.edu.cn/">Wuhan
University</a>, <a href="http://maths.whu.edu.cn/">School
of mathematics
and statistics </a>, Received the master's degree.<br>

&bull;&nbsp;1999~2003 <a href="http://www.whu.edu.cn/">Wuhan
University</a>, <a href="http://maths.whu.edu.cn/">School
of mathematics
and statistics </a>, Received the bachelor's degree.</span></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE8">Research
Interests:</span><br>

      <span class="STYLE4">&nbsp;&nbsp;Bioinformatics,
Computational systems biology,
Machine learning, Graph theory, Precision medicine</span></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE8">Research
and professional experence:</span><br>

      <span class="STYLE4">&bull;&nbsp;2014/12~now,
      <strong>associate Professor</strong>, <a href="http://www.sysbio.ac.cn/">Key Laboratory
of Systems Biology </a>, <a href="http://www.sibs.cas.cn/">Shanghai
Institutes for Biological Sciences </a>, <a href="http://english.cas.cn/">Chinese Academy of
Sciences </a><br>

&bull;&nbsp;2013/1-2014/11, <strong>assistant Professor</strong>,
      <a href="http://www.sysbio.ac.cn/">Key Laboratory
of Systems Biology </a>, <a href="http://www.sibs.cas.cn/">Shanghai
Institutes for Biological Sciences </a>, <a href="http://english.cas.cn/">Chinese Academy of
Sciences </a><br>

&bull;&nbsp;2011/1-2012/12, <strong>post-doctor</strong>,
      <a href="http://www.sysbio.ac.cn/">Key Laboratory
of Systems Biology </a>, <a href="http://www.sibs.cas.cn/">Shanghai
Institutes for Biological Sciences </a>, <a href="http://english.cas.cn/">Chinese Academy of
Sciences </a><br>

      </span><span class="STYLE4">&bull;&nbsp;2008/7-2009/12,
      <strong>research-associate</strong>, <a href="http://www3.ntu.edu.sg/birc/">BioInformatics
Research Centre </a>, <a href="http://scse.ntu.edu.sg/Pages/Home.aspx">
School of Computer Engineering </a>, <a href="http://www.ntu.edu.sg/Pages/home.aspx">
Nanyang Technological
University </a>, Singapore</span></p>

      <p class="STYLE10"></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE8">Academic
activities:</span><br>

      <span class="STYLE4">&bull;&nbsp;The
academic secretary of <a href="http://www.xssc.ac.cn/xs/showconf.asp?pid=455&amp;tid=4">The
557th Session of XIANGSHAN-SCIENCE Conferences: Big Data, Precision
Medicine and Bioinformatics</a> <br>

&bull;&nbsp;The
organizing chair of <a href="http://sysbio.sibcb.ac.cn/cb/BioSynergy2015.html">Bio-synergy
2015</a> <br>

&bull;&nbsp;The report on Highlights sessions of <a href="http://www.aporc.org/ISB/2015/">ISB 2015</a> <br>

&bull;&nbsp;The
invited report at Shanghai Bioinformatics Seminar (2016)<br>

&bull;&nbsp;The
invited report at School of Mathematics, South China University of
Technology (2016)<br>

&bull;&nbsp;The reviewers for peer-review journals, such as
TCBB, JTB, AIIM, SREP, BMC Genomics, BMC Cancer, BMC Systems Biology,
CBM, MBS, Methods, IET SYB, IJBRA, IJMI, IJDMB, etc <br>

&bull;&nbsp;The program committee member of <a href="http://www.aporc.org/ISB/2013/">ISB 2013</a>
, <a href="http://www.aporc.org/ISB/2014/">ISB 2014</a>
, <a href="http://www.aporc.org/ISB/2015/">ISB 2015</a>
, <a href="http://www.aporc.org/ISB/2016/">ISB 2016</a>
, <a href="http://www.aporc.org/ISB/2017/">ISB 2017</a>
, <a href="http://datamining-web.it.uts.edu.au/bioinformatics/committees.html">InCob
2017</a> , <a href="http://ic-ic.tongji.edu.cn/2018/index.htm">ICIC
2018</a> </span></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE1"><span class="STYLE9">Projcts:</span><br>

      </span><span class="STYLE4">&bull;&nbsp;NSFC
(31200987). (PI) <br>

&bull;&nbsp;The Knowledge Innovation Program of SIBS CAS
(2013KIP218). (PI) <br>

&bull;&nbsp;The Natural Science Foundation of Shanghai
(17ZR1446100). (PI) <br>

&bull;&nbsp;The Strategic Priority Research Program of CAS
(XDB13040700). <br>

&bull;&nbsp;The National Key R&amp;D Program - Special
Project on Precision Medicine (2016YFC0903400)). <br>

&bull;&nbsp;The National key research and development program
of China (No. 2017YFA0505500). <br>

&bull;&nbsp;Project from Shanghai cooperative center for
transformation &amp; innovation. (Co-PI) <br>

      </span></p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE1"><span class="STYLE9"><span class="STYLE8">Teaching
area and courses</span>:</span><br>

      </span><span class="STYLE4">Bioinformatics,
2013~2017 in SIBS</span>.</p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><span class="STYLE11">Publication:</span></p>

      <p class="STYLE10"><em>Journal paper:</em><br>

-------------------------------------------------------2017-----------------------------------------------------<br>

      <span class="STYLE12">[1] Yu X, Zhang J, Sun S, Zhou
X, <strong>Zeng T</strong>, Chen L. Individual-specific
edge-network analysis for disease prediction. Nucleic Acids Res. 2017
Nov 16;45(20):e170. doi: 10.1093/nar/gkx787. PubMed
PMID: 28981699. (Co-corresponding author) </span></p>

      <p class="STYLE3">[2] Shi Q, Zhang C, Peng M, Yu X, <strong>Zeng
T</strong>, Liu J, Chen L. Pattern fusion analysis by adaptive
alignment of multiple heterogeneous omics data. Bioinformatics. 2017
Sep 1;33(17):2706-2714. doi: 10.1093/bioinformatics/btx176. PubMed
PMID: 28520848. (Co-corresponding author)</p>

      <p class="STYLE3">[3] Zhang C, Liu J, Shi Q, <strong>Zeng
T</strong>
, Chen L. Comparative network stratification analysis for identifying
functional interpretable network biomarkers. BMC Bioinformatics. 2017
Mar 14;18(Suppl 3):48. doi: 10.1186/s12859-017-1462-x. PubMed
PMID: 28361683. (Co-corresponding author) </p>

      <p class="STYLE3">[4] Li M, Li C, Liu WX, Liu C, Cui
J, Li Q, Ni H, Yang Y, Wu C, Chen C, Zhen X, <strong>Zeng
T</strong>
, Zhao M, Chen L, Wu J, Zeng R, Chen L. Dysfunction of PLA2G6 and
CYP2C44-associated network signals imminent carcinogenesis from chronic
inflammation to hepatocellular carcinoma. J Mol Cell Biol. 2017 Jun 26.
doi: 10.1093/jmcb/mjx021. [Epub ahead of print] PubMed
PMID: 28655161. </p>

      <p class="STYLE3">[5] Shi Q, Zhang C, Guo W, <strong>Zeng
T</strong>
, Lu L, Jiang Z, Wang Z, Liu J, Chen L. Local network component
analysis for quantifying transcription factor activities. Methods. 2017
Jul 15;124:25-35. doi: 10.1016/j.ymeth.2017.06.018. Epub 2017 Jul 12.
PubMed
PMID: 28710010. )</p>

      <p class="STYLE10">-------------------------------------------------------2016-----------------------------------------------------<br>

      <span class="STYLE12">[1] Yang Y, Lin X, Lu X, Luo
G, <strong>Zeng T</strong>, Tang J, Jiang F, Li L, Cui X,
Huang W, Hou G, Chen X, Ouyang Q, Tang S, Sun H, Chen L, Gonzalez FJ,
Wu M, Cong W, Chen L, Wang H. Interferon-microRNA signalling drives
liver precancerous lesion formation and hepatocarcinogenesis. Gut. 2016
Feb 9. pii: gutjnl-2015-310318. doi: 10.1136/gutjnl-2015-310318. [Epub
ahead of print] PubMed PMID: 26860770.</span></p>

      <p class="STYLE10">-------------------------------------------------------2015-----------------------------------------------------<br>

      <span class="STYLE12">[1] <strong>Zeng T</strong>,
Zhang W, Yu X, Liu X, Li M, Chen L. Big-data-based edge biomarkers:
study on dynamical drug sensitivity and resistance in individuals.
Brief Bioinform. 2015 Sep 26. pii: bbv078. [Epub ahead of print] PubMed
PMID: 26411472.</span></p>

      <p class="STYLE3">[2] Yu X, <strong>Zeng T</strong>,
Li G. Integrative enrichment analysis: a new computational method to
detect dysregulated pathways in heterogeneous samples. BMC Genomics.
2015 Nov 10;16:918. doi: 10.1186/s12864-015-2188-7. PubMed PMID:
26556243; PubMed Central PMCID: PMC4641376. (Co-corresponding author)</p>

      <p class="STYLE3">[3] Kang H, Cho KH, Zhang XD, <strong>Zeng
T</strong>, Chen L. Inferring Sequential Order of Somatic
Mutations during Tumorgenesis based on Markov Chain Model. IEEE/ACM
Trans Comput Biol Bioinform. 2015 Sep-Oct;12(5):1094-103. doi:
10.1109/TCBB.2015.2424408. PubMed PMID: 26451822. (Co-corresponding
author)</p>

      <p class="STYLE3">[4] Yu X, <strong>Zeng T</strong>,
Wang X, Li G, Chen L. Unravelling personalized dysfunctional gene
network of complex diseases based on differential network model. J
Transl Med. 2015 Jun 13;13:189. doi: 10.1186/s12967-015-0546-5. PubMed
PMID: 26070628; PubMed Central PMCID: PMC4467679.</p>

      <p class="STYLE3">[5] Shi Q, Liu X, <strong>Zeng
T</strong>, Wang W, Chen L. Detecting disease genes of non-small
lung cancer based on consistently differential interactions. Cancer
Metastasis Rev. 2015 Jun;34(2):195-208. doi: 10.1007/s10555-015-9561-5.
PubMed PMID: 26004969.</p>

      <p class="STYLE3">[6] Yang D, Zhou J,<strong>
Zeng T</strong>, Yang Z, Wang X, Hu J, Song Y, Chen L, Peer D,
Wang X, Bai C. Serum chemokine network correlates with chemotherapy in
non-small cell lung cancer. Cancer Lett. 2015 Aug 28;365(1):57-67. doi:
10.1016/j.canlet.2015.05.001. Epub 2015 May 11. PubMed PMID: 25976768.</p>

      <p class="STYLE3">[7] Chen S, Liu J, <strong>Zeng
T</strong>. Measuring the quality of linear patterns in
biclusters. Methods. 2015 Jul 15;83:18-27. doi:
10.1016/j.ymeth.2015.04.005. Epub 2015 Apr 15. PubMed PMID: 25890245.</p>

      <p class="STYLE3">[8] Zhang W, <strong>Zeng T</strong>,
Liu X, Chen L. Diagnosing phenotypes of single-sample individuals by
edge biomarkers. J Mol Cell Biol. 2015 Jun;7(3):231-41. doi:
10.1093/jmcb/mjv025. Epub 2015 Apr 26. PubMed PMID: 25917597. (Co-first
author)</p>

      <p class="STYLE10">-------------------------------------------------------2014-----------------------------------------------------<br>

      <span class="STYLE12">[1] <strong>Zeng T</strong>,
Zhang W, Yu X, Liu X, Li M, Liu R, Chen L. Edge biomarkers for
classification and prediction of phenotypes. Sci China Life Sci. 2014
Nov;57(11):1103-14. doi: 10.1007/s11427-014-4757-4. Epub 2014 Oct 17.
Review. PubMed PMID: 25326072.</span></p>

      <p class="STYLE3">[2] Wen Z, Zhang W,<strong>
Zeng T</strong>, Chen L. MCentridFS: a tool for identifying
module biomarkers for multi-phenotypes from high-throughput data. Mol
Biosyst. 2014 Nov;10(11):2870-5. doi: 10.1039/c4mb00325j. PubMed PMID:
25099602.</p>

      <p class="STYLE3">[3] <strong>Zeng T</strong>,
Wang DC, Wang X, Xu F, Chen L. Prediction of dynamical drug sensitivity
and resistance by module network rewiring-analysis based on
transcriptional profiling. Drug Resist Updat. 2014 Jul;17(3):64-76.
doi: 10.1016/j.drup.2014.08.002. Epub 2014 Aug 13. Review. PubMed PMID:
25156319.</p>

      <p class="STYLE3">[4] Zhang W, <strong>Zeng T</strong>,
Chen L. EdgeMarker: Identifying differentially correlated molecule
pairs as edge-biomarkers. J Theor Biol. 2014 Dec 7;362:35-43. doi:
10.1016/j.jtbi.2014.05.041. Epub 2014 Jun 12. PubMed PMID: 24931676.
(Co-corresponding author)</p>

      <p class="STYLE3">[5] <strong>Zeng T</strong>,
Zhang CC, Zhang W, Liu R, Liu J, Chen L. Deciphering early development
of complex diseases by progressive module network. Methods. 2014 Jun
1;67(3):334-43. doi: 10.1016/j.ymeth.2014.01.021. Epub 2014 Feb 21.
PubMed PMID: 24561825.</p>

      <p class="STYLE3">[6] Zhao M, Chang HT, Zhou Q, <strong>Zeng
T</strong>, Shih CS, Liu ZP, Chen L, Wei DQ. Predicting
protein-ligand interactions based on chemical preference features with
its application to new D-amino acid oxidase inhibitor discovery. Curr
Pharm Des. 2014;20(32):5202-11. PubMed PMID: 24410568.</p>

      <p class="STYLE3">[7] Li M, <strong>Zeng T</strong>,
Liu R, Chen L. Detecting tissue-specific early warning signals for
complex diseases based on dynamical network biomarkers: study of type 2
diabetes by cross-tissue analysis. Brief Bioinform. 2014
Mar;15(2):229-43. doi: 10.1093/bib/bbt027. Epub 2013 Apr 25. PubMed
PMID: 23620135. (Co-first author)</p>

      <p class="STYLE10">------------------------------------------------2013
and before-------------------------------------------------<br>

      <span class="STYLE12">[1]<strong> Zeng T</strong>,
Sun SY, Wang Y, Zhu H, Chen L. Network biomarkers reveal dysfunctional
gene regulations during disease progression. FEBS J. 2013
Nov;280(22):5682-95. doi: 10.1111/febs.12536. Epub 2013 Oct 22. Review.
PubMed PMID: 24107168.</span></p>

      <p class="STYLE3">[2] Sun SY, Liu ZP, <strong>Zeng
T</strong>, Wang Y, Chen L. Spatio-temporal analysis of type 2
diabetes mellitus based on differential expression networks. Sci Rep.
2013;3:2268. doi: 10.1038/srep02268. PubMed PMID: 23881262; PubMed
Central PMCID: PMC3721080.</p>

      <p class="STYLE3">[3] <strong>Zeng T</strong>,
Chen L. Tracing dynamic biological processes during phase transition.
BMC Syst Biol. 2012;6 Suppl 1:S12. doi: 10.1186/1752-0509-6-S1-S12.
Epub 2012 Jul 16. Erratum in: BMC Syst Biol. 2012 Jul 16;6 Suppl 1:S23.
PubMed PMID: 23046764; PubMed Central PMCID: PMC3403121.</p>

      <p class="STYLE3">[4] Xiong Y, Liu J, Zhang W, <strong>Zeng
T</strong>. Prediction of heme binding residues from protein
sequences with integrative sequence profiles. Proteome Sci. 2012 Jun
21;10 Suppl 1:S20. doi: 10.1186/1477-5956-10-S1-S20. PubMed PMID:
22759579; PubMed Central PMCID: PMC3380730.</p>

      <p class="STYLE3">[5]<strong> Zeng T</strong>,
Li J, Liu J. Distinct interfacial biclique patterns between
ssDNA-binding proteins and those with dsDNAs. Proteins. 2011
Feb;79(2):598-610. doi: 10.1002/prot.22908. PubMed PMID: 21120860.</p>

      <p class="STYLE3">[6]<strong> Zeng T</strong>,
Li J. Maximization of negative correlations in time-course gene
expression data for enhancing understanding of molecular pathways.
Nucleic Acids Res. 2010 Jan;38(1):e1. doi: 10.1093/nar/gkp822. Epub
2009 Oct 23. PubMed PMID: 19854949; PubMed Central PMCID: PMC2800212.</p>

      <p class="STYLE3">[7] <strong>Zeng T</strong>,
Liu J. Mixture classification model based on clinical markers for
breast cancer prognosis. Artif Intell Med. 2010 Feb-Mar;48(2-3):129-37.
doi: 10.1016/j.artmed.2009.07.008. Epub 2009 Dec 14. PubMed PMID:
20005686.</p>

      <p class="STYLE3">[8] Gu C, <strong>Zeng T</strong>,
Li Y, Xu Z, Mo Z, Zheng C. Structure-function analysis of mutant
RNA-dependent RNA polymerase complexes with VPg. Biochemistry (Mosc).
2009 Oct;74(10):1132-41. PubMed PMID: 19916926. (Co-first author) </p>

      <p class="STYLE3">[9] <strong>Zeng T</strong>,
Zhongxi Mo. Bioinformatics analysis of mutant structure and functional
sites of foot-and-mouth disease virus RNA-dependent RNA polymerase.
Journal of Computational and Theoretical Nanoscience, v 4, n 7-8, p
1330-1336, November/December 2007. </p>

      <p class="STYLE3">[10] Zhongxi Mo, <strong>Zeng
T</strong> . Improved genetic algorithm for problem of genome
rearrangement. Wuhan University Journal of Natural Sciences, v 11, n 3,
p 498-502, May 2006.<br>

      </p>

      <p class="STYLE3">&nbsp;</p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE10"><em>Conference paper:</em><br>

      <span class="STYLE12">
[1] <strong>Tao Zeng</strong>. Improving the
discrimination and interpretability for complex diseases by network
biomarker. 16th SCBA International Symposium, 2017, pp.298. (Abstract)
      </span></p>

      <p class="STYLE3"> [2] Weifeng Guo, Shaowu Zhang,
Qianqian Shi, Chengming Zhang, <strong>Tao Zeng</strong>
and Luonan Chen . A novel algorithm for finding optimal driver nodes to
target control complex networks and its applications for drug targets
identification. International Conference on Bioinformatics (InCoB)
2017.
      </p>

      <p class="STYLE3">[3] Chuanchao Zhang, Juan Liu, Qianqian Shi, <strong>Tao Zeng</strong> and Luonan Chen.
Comparative network stratification analysis for identifying functional
interpretable network biomarkers. The Asia Pacific Bioinformatics
Conference (APBC) 2017.
      </p>

      <p class="STYLE3"> [4] Chuanchao Zhang, Juan Liu, <strong>Tao
Zeng</strong>, and Luonan Chen. Integration of multiple
heterogeneous omics data
.The IEEE International Conference on Bioinformatics and Biomedicine
(BIBM) 2016.
      </p>

      <p class="STYLE3">[5] Chuanchao Zhang, Juan Liu,
Qianqian Shi, <strong>Tao Zeng</strong>, Luonan Chen.
Identification of phenotypic networks based on whole transcriptome by
comparative network decomposition. 2015 IEEE International Conference
on Bioinformatics and Biomedicine (BIBM), pp: 189-194.
      </p>

      <p class="STYLE3">[6] <strong>Tao Zeng</strong>,
Ziming Wang, Luonan Chen: Detecting Phosphorylation Determined Active
Protein Interaction Network during Cancer Development by Robust Network
Component Analysis. DTMBIO '14 Proceedings of the ACM 8th International
Workshop on Data and Text Mining in Bioinformatics ,DTMBIO@CIKM 2014,
pp.49. (Abstract)</p>

      <p class="STYLE3">[7] <strong>Tao Zeng</strong>,
Ruozhao Wang, Luonan Chen. Module network rewiring in response to
therapy. IEEE 6th International Conference on Systems Biology, ISB
2012, p 197-202.</p>

      <p class="STYLE3">[8] <strong>Tao Zeng</strong>,
Luonan Chen. Identifying temporal trace of biological process during
phase transition. Systems Biology (ISB), 2011 IEEE International
Conference on, pp:368-373.</p>

      <p class="STYLE3">[9] Guanying Piao, Bangguo Qian,
Shigeru Saito, ZP Liu, <strong>Tao Zeng</strong>, Y. Wang,
Jiarui Wu, Huarong Zhou, L. Chen, K. Horimoto. Phenotype-difference
oriented identification of molecular functions for diabetes progression
in Goto-Kakizaki rat. Systems Biology (ISB), 2011 IEEE International
Conference on, pp:111-116.</p>

      <p class="STYLE3">[10] Yi Xiong, Wen Zhang, <strong>Tao
Zeng</strong>, Juan Liu. Prediction of heme binding sites in heme
proteins using an integrative sequence profile coupling evolutionary
information with physicochemical properties. Proceedings - 2011 IEEE
International Conference on Bioinformatics and Biomedicine, BIBM 2011,
p 143-146, 2011.</p>

      <p class="STYLE3">[11] Jinyan Li, Qian Liu, <strong>Tao
Zeng</strong>. Negative correlations in collaboration: concepts
and algorithms. Knowledge Discovery and Data Mining, KDD2010, p
463-472, 2010.</p>

      <p class="STYLE3">[12] <strong>Tao Zeng</strong>,
Xuan Guo, Juan Liu. Discovering negative correlated gene sets from
integrative gene expression data for cancer prognosis. Proceedings -
2010 IEEE International Conference on Bioinformatics and Biomedicine,
BIBM 2010, p 489-492, 2010, </p>

      <p class="STYLE3">[12] <strong>Tao Zeng</strong>,
Fei Luo, Juan Liu. Biological pathway conducting microarray-based
cancer classification. Proceedings of the 2009 2nd International
Conference on Biomedical Engineering and Informatics, BMEI 2009, 2009.</p>

      <p class="STYLE3">[13] <strong>Tao Zeng</strong>,
Juan Liu. A novel mixture classifier and its application in breast
cancer prognosis. 2nd International Conference on Bioinformatics and
Biomedical Engineering, iCBBE 2008, p 564-568, 2008.</p>

      <p class="STYLE3">[14] <strong>Tao Zeng</strong>,
Juan Liu. Analysis on time-lagged gene clusters in time series gene
expression data. Proceedings - 2007 International Conference on
Computational Intelligence and Security, CIS 2007, p 182-185, 2007.</p>

      <p class="STYLE10">&nbsp;</p>

      <p class="STYLE8">Softwares:</p>

      <p class="STYLE4">&hearts;&nbsp;MCentridFS (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/MCentridFS.rar">Multi-class
Centroid Feature Selection</a>): supply a tool for the optimal
module analysis on the biological active network </p>

      <p class="STYLE4">&hearts;&nbsp;PMNminer (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/PMNminer.rar">Progressive
module network analysis</a>): supply a computational framework
for the cross-tissue analysis on the phenotype-dynamics of biological
process (e.g. disease occurance and progression) </p>

      <p class="STYLE4">&hearts;&nbsp;IEApackage (<a href="https://github.com/bluesky2009/integrative-enrichment-analysis">Integrative
enrichment analysis</a>): supply a software package for the
functional enrichment analysis on the heterogeneous samples </p>

      <p class="STYLE4">&hearts;&nbsp;PFA (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/PFApackage_0.1.rar">Pattern
fusion analysis</a>): supply an integrative tool to identify
sample-patterns from different omics profiles by optimally adjusting
the effects of each data type to the hidden patterns. </p>

      <p class="STYLE4">&hearts;&nbsp;CNS (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/CNSpackage_0.1.rar">Comparative
Network Stratification</a>): supply a computational methods to
directly identify functional network biomarkers with both
discriminative power on disease states and readable interpretation on
biological functions. </p>

      <p class="STYLE4">&hearts;&nbsp;DFApackage (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/DFApackage.zip">Differential
function analysis</a>): supply a general framework to identify
the changes of element-structure and expression-activation of
biological functions, based on comparative non-negative matrix
factorization (cNMF). </p>

      <p class="STYLE4">&hearts;&nbsp;SCS (<a href="http://sysbio.sibcb.ac.cn/cb/chenlab/images/supplemntary_MATERIAL.zip">Single-sample
Controller Strategy</a>): supply a new single-sample controller
strategy (SCS) to identify personalized driver mutation profiles from
network controllability perspective. </p>

      </td>

    </tr>

  </tbody>
</table>

<br>

<!-- bottom menu table -->
<!-- bottom menu table --><font face="Verdana" size="1">&copy;2016 Homepage of Tao Zeng <b>SIBS&trade;</b></font><br>

</div>

<span style="display: none;"><a href="http://www.mianfeimoban.com">mianfeimoban</a></span>
</body>
</html>
